tools/cargo_gn_test.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+# This script is invoked independently of tools/test.py.
+# TestCargoGn is purposely excluded from the main tests (tools/test.py) as it
+# invokes a new build. It can be included if build.py is removed and "cargo
+# build" becomes the main frontend.
+# https://github.com/denoland/deno/issues/2608
+
+import os
+import unittest
+from util import run_output
+
+
+def touch(fname, times=None):
+    with open(fname, 'a'):
+        os.utime(fname, times)
+
+
+# Warning: To avoid expensive rebuilds in CI, this should match the command
+# invoked in travis and appveyor.
+CARGO_CMD = ["cargo", "build", "-vv", "--release", "--locked"]
+
+
+# Tests for cargo-gn integration.
+class TestCargoGn(unittest.TestCase):
+    def test_rerun_if_changed(self):
+        # Here we are checking that "cargo build" does proper incremental
+        # compilation. First we must run "cargo build" once, in case it hasn't
+        # been done yet.
+        print "[1/3] cargo build"
+        result1 = run_output(CARGO_CMD, quiet=True)
+        self.assertEqual(result1.code, 0)
+        assert "Finished" in result1.err
+
+        # Now we run it again, checking that there aren't any
+        # cargo:rerun-if-changed lines in the output.
+        print "[2/3] cargo build again..."
+        result2 = run_output(CARGO_CMD, quiet=True)
+        self.assertEqual(result2.code, 0)
+        assert "Finished" in result2.err
+        assert "cargo:rerun-if-changed" not in result2.out
+
+        # Finally we touch some file that the build ought to depend on,
+        # and check that the build does get rebuilt.
+        touch("js/main.ts")
```

Comment:
This working correctly is critical, so we should perform this test for some more files that are not explicitly listed in gn. E.g.
`ts-library-builder.js'
'node_modules/typescript/index.js'
'msg.fbs'
'third_party/v8/include/v8-config.h'

Reply:
Doing this properly adds about ~2 minutes per file to appveyor build time. I think just doing it for one file is sufficient for now.

Reply:
> Doing this properly adds about ~2 minutes per file to appveyor build time. I think just doing it for one file is sufficient for now.

So did you check that it does indeed work correctly for the files I listed? Either way, the file you picked is very easy to detect as a dependency. Pick something that's hard.

Reply:
I don't think it works for node_modules, it does work for v8 header files.

Reply:
here is the list of files we rerun build.rs on:

```
[arrayvec 0.4.10] cargo:rerun-if-changed=build.rs
[proc-macro2 0.4.30] cargo:rerun-if-changed=build.rs
[deno 0.10.0] cargo:rerun-if-env-changed=DENO_BUILD_PATH
[deno 0.10.0] cargo:rerun-if-env-changed=APPVEYOR_REPO_COMMIT
[rand_pcg 0.1.2] cargo:rerun-if-changed=build.rs
[rand_chacha 0.1.1] cargo:rerun-if-changed=build.rs
[rand 0.6.5] cargo:rerun-if-changed=build.rs
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/code_generators.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../testing/gtest/include/gtest/gtest_prod.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_lua.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_personality.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/support/runtime/stdexcept_default.ipp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/fallback_malloc.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/crypto.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/mutex.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/optional.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/modes/internal.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/grpc/src/compiler/cpp_generator.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/diagnostics/crash-key.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/abort_message.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/reflection.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_vector.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/include/GFp/type_check.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz384.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_handlers.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest-spi.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/chacha-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/bn/montgomery.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/internal.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/lazy-instance.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/template-utils.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/custom/gtest/internal/custom/gtest-port.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/variant.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/gtest-internal.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/include/refstring.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/bn/internal.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/grpc/src/compiler/go_generator.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/vector.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/platform.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/file-utils.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/grpc/src/compiler/config.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/flexbuffers.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/gfp_p384.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/poly1305-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest-matchers.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_default_handlers.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/stdlib_exception.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_handlers.hpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/sha256-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz384.inl
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/reflection.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest-printers.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/flatc_main.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/new.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/system_error.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/string.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/division-by-constant.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/cpu-intel.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/future.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/limbs/limbs.inl
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/limbs/limbs.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest_pred_impl.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_exception_storage.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest-printers.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_lobster.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/code_generators.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/ieee754.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest-param-test.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/flatc.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/debug/stack_trace.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/v8-fallthrough.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/flatc.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/abort_message.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/logging.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/stdlib_typeinfo.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/hash.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/custom/gtest.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/support/runtime/exception_libcxxabi.ipp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_exception.hpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/p256-x86_64-asm-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/mutex.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/modes/gcm.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/gtest-type-util.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_general.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/grpc/src/compiler/go_generator.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/random.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_grpc.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/demangle/ItaniumDemangle.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/limbs/limbs.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/support/runtime/exception_pointer_cxxabi.ipp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../testing/gtest/empty.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_dart.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_python.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/file-utils.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/compiler-specific.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz256_table.inl
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/aesni-gcm-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/build_config.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/aesni-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/private_typeinfo.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/include/apple_availability.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-string-conversions.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/overflowing-math.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_go.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_unexpected.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/bits.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/once.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/regex.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/bits.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_php.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/export-template.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/any.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8-platform.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest-typed-test.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/block.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/bn/generic.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/gfp_p256.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/typeinfo.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/x86_64-mont-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/private_typeinfo.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest-death-test.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/fallback_malloc.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest-filepath.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/bounded-page-allocator.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_parser.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/flatbuffers.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz256.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest-port.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/debug.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/bind.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/mem.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/semaphore.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/grpc/src/compiler/cpp_generator.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/stdexcept.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/stdlib_stdexcept.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/ios.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/include/GFp/base.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_text.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/grpc/src/compiler/java_generator.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/include/GFp/cpu.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/base-export.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_js.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/grpc/src/compiler/java_generator.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/algorithm.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/hash.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/third_party/inspector_protocol/encoding/encoding.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/region-allocator.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/block.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/third_party/inspector_protocol/encoding/encoding.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/functional.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/address-region.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/third_party/fiat/curve25519.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/functional.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/demangle/StringView.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_exception.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest-typed-test.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/condition_variable.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_rust.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/utility.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/functional.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/x86_64-mont5-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/ghash-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/idl.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/cpu.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/cpu.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/stl_emulation.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/base.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/grpc/src/compiler/schema_interface.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_json_schema.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest-message.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/third_party/fiat/internal.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz256.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_fbs.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/idl_gen_cpp.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_demangle.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/diagnostics/crash-key-noop.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/include/GFp/mem.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/gtest-string.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/bounded-page-allocator.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/utils/random-number-generator.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/src/util.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_guard.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest-death-test.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/demangle/DemangleConfig.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/memory.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/include/atomic_support.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/chrono.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/vpaes-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/aes-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/include/refstring.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/gtest-filepath.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/debug/stack_trace.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/thread.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/third_party/fiat/curve25519_tables.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/include/config_elast.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest-test-part.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/locale.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest-matchers.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/include/atomic_support.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/gtest-port-arch.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_aux_runtime.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/iostream.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/strstream.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/once.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/division-by-constant.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/exception.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/demangle/Utility.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/ieee754.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/src/gtest-internal-inl.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-string-conversions.cc
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/valarray.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/gtest-param-util.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/bn/montgomery_inv.c
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8config.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/custom/gtest-printers.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/shared_mutex.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/util.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_virtual.cpp
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/macros.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/flatbuffers/include/flatbuffers/reflection_generated.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/sha512-x86_64-macosx.S
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/internal/gtest-death-test-internal.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest-test-part.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest_prod.h
[deno 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/logging.h
```

Reply:
Here is a list of the files it detects:

```
[arrayvec 0.4.10] cargo:rerun-if-changed=build.rs
[proc-macro2 0.4.30] cargo:rerun-if-changed=build.rs
[deno 0.10.0] cargo:rerun-if-env-changed=DENO_BUILD_PATH
[deno 0.10.0] cargo:rerun-if-env-changed=APPVEYOR_REPO_COMMIT
[rand_pcg 0.1.2] cargo:rerun-if-changed=build.rs
[rand_chacha 0.1.1] cargo:rerun-if-changed=build.rs
[rand 0.6.5] cargo:rerun-if-changed=build.rs
[deno_cli 0.10.0] cargo:rerun-if-env-changed=DENO_BUILD_PATH
[deno_cli 0.10.0] cargo:rerun-if-env-changed=APPVEYOR_REPO_COMMIT
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/cli/gn_args.txt
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/assets.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/blob.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/body.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/buffer.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/build.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/chmod.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/chown.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/colors.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/compiler.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/console.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/console_table.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/copy_file.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/core.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/custom_event.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/deno.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/diagnostics.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/dir.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/dispatch.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/dispatch_minimal.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/dom_types.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/dom_util.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/errors.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/event.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/event_target.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/fetch.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/format_error.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/dom_file.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/file_info.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/files.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/flatbuffers.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/form_data.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/get_random_values.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/globals.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/headers.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/io.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/lib.web_assembly.d.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/link.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/location.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/main.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/make_temp_dir.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/metrics.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/mkdir.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/mock_builtin.js
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/net.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/os.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/permissions.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/plugins.d.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/process.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/read_dir.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/read_file.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/read_link.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/remove.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/rename.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/repl.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/request.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/resources.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/stat.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/symlink.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/text_encoding.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/timers.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/truncate.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/types.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/url.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/url_search_params.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/util.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/utime.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/window.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/workers.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/write_file.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/performance.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/version.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../js/xeval.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../tsconfig.json
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../package.json
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/yarn.lock
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../tools/ts_library_builder/tsconfig.json
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../tools/ts_library_builder/main.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../tools/ts_library_builder/build_library.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../tools/ts_library_builder/ast_util.ts
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/cli/gn_args.txt
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/regex.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/libplatform/libplatform-export.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/default-worker-threads-task-runner.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/include/atomic_support.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/injected-script.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-debugger-agent-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/task-queue.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/init/v8.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Runtime.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/support/runtime/stdexcept_default.ipp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/inspected-context.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Profiler.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/sys-info.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/custom-preview.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/atomic-utils.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/file-utils.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-debugger.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/once.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/atomicops_internals_atomicword_compat.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/string-16.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-schema-agent-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/hash.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/exception.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/future.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/fallback_malloc.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-stack-trace-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/ieee754.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/bind.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/platform.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/include/apple_availability.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-console-message.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/v8-fallthrough.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/default-worker-threads-task-runner.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/diagnostics/crash-key-noop.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/time.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8config.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/optional.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_handlers.hpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/lazy-instance.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_exception.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/cfg.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/stdexcept.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/once.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/debug/stack_trace.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/cpu.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/injected-script.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/ios.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/division-by-constant.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/common/checks.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/base-export.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/numbers/conversions.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/libplatform/v8-tracing.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/vector.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/diagnostics/crash-key.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/include/inspector/Runtime.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/overflowing-math.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/inspected-context.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/functional.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/earley-parser.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/declarable.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/torque-compiler.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Debugger.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/bounded-page-allocator.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/chrono.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/abort_message.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/common/v8memory.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/semaphore.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/algorithm.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libsampler/sampler.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/region-allocator.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/utils/random-number-generator.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Console.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/default-platform.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/logging.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/torque-parser.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/delayed-task-queue.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/instructions.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/macros.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/third_party/inspector_protocol/encoding/encoding.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/support/runtime/exception_libcxxabi.ipp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/include/refstring.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/platform-posix.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Debugger.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/condition-variable.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-value-utils.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/default-foreground-task-runner.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/types.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/time.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/file-utils.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/debug.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/instructions.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-profiler-agent-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/cfg.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/test-interface.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/random.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-inspector-protocol-encoding.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/tracing/trace-buffer.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Protocol.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/ast.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/sys-info.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/ieee754.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-inspector-protocol-encoding.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Forward.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/debug/debug-interface.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/tracing/trace-writer.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_vector.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-regex.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/include/atomic_support.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/safe_conversions_impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/address-region.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/utils/random-number-generator.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/safe_conversions.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/region-allocator.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/remote-object-id.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/worker-thread.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../testing/gtest/include/gtest/gtest_prod.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/compiler-specific.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libsampler/sampler.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/tracing/tracing-controller.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/inspector_jumbo_1.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/constants.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/tracing/trace-event.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/string.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_exception_storage.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/declarations.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/task-queue.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-schema-agent-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/types.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/include/config_elast.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/abort_message.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/page-allocator.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/libplatform/libplatform.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/memory.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8-profiler.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-string-conversions.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/implementation-visitor.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_default_handlers.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/wasm-translation.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-inspector-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/remote-object-id.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/string-util.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/search-util.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/debug/stack_trace_posix.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Runtime.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/include/inspector/Schema.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/include/inspector/Debugger.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/torque-compiler.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/torque-parser.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/value-mirror.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/platform-posix.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_personality.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/strstream.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/custom-preview.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/support/runtime/exception_pointer_cxxabi.ipp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-console-agent-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/free_deleter.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/type-oracle.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/utils/vector.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-debugger-script.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-console.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/value-mirror.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/source-positions.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/declaration-visitor.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-profiler-agent-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/mutex.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_handlers.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/torque.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/bits.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-inspector-session-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_exception.hpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_aux_runtime.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/page-allocator.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/contextual.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8-version.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/iostream.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/type-oracle.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/flags.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-debugger-script.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/export-template.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/tracing/trace-buffer.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Schema.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/debug/stack_trace.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/utils/allocation.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/csa-generator.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-console-agent-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/platform-macos.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-heap-profiler-agent-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_unexpected.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/string-16.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/system_error.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/semaphore.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/optional.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/elapsed-timer.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/any.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/shared_mutex.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/default-foreground-task-runner.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Schema.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/HeapProfiler.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-runtime-agent-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Protocol.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/torque_base_jumbo_1.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/thread.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/server-data.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/tracing/trace-config.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/test-interface.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/wasm-translation.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/template-utils.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-string-conversions.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-console-message.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/declarable.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/source-positions.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-debugger.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/private_typeinfo.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/third_party/inspector_protocol/encoding/encoding.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-console.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-stack-trace-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/declarations.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/worker-thread.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/type-visitor.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Profiler.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/atomicops.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-inspector-session-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/utility.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/mutex.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/functional.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/typeinfo.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-heap-profiler-agent-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/valarray.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/earley-parser.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8-internal.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/bounded-page-allocator.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-runtime-agent-impl.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/condition_variable.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8-inspector.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/declaration-visitor.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/timezone-cache.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/csa-generator.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++abi/trunk/src/cxa_virtual.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/server-data.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/default-platform.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/variant.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/build_config.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/cpu.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/division-by-constant.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/implementation-visitor.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8-platform.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/atomicops_internals_portable.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/mutex.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/HeapProfiler.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/global-context.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/tracing/trace-writer.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/common/globals.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/base/trace_event/common/trace_event_common.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/utils.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/string-util.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/platform-posix-time.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/bits.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/delayed-task-queue.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/debug/interface-types.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/condition-variable.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/platform/platform-posix-time.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/type-visitor.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/utils.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/libplatform/tracing/trace-object.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/new.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/googletest/src/googletest/include/gtest/gtest_prod.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-value-utils.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-inspector-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/locale.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/search-util.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/torque/parameter-difference.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/include/v8-util.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/base/logging.h
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-debugger-agent-impl.cc
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/gen/third_party/v8/src/inspector/protocol/Console.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../buildtools/third_party/libc++/trunk/src/functional.cpp
[deno_cli 0.10.0] cargo:rerun-if-changed=/Users/rld/src/deno/target/debug/../../third_party/v8/src/inspector/v8-regex.h

```

Reply:
Fewer files than I'd expect.


```
Gallivant:deno piscisaureus$ third_party/depot_tools/ninja -C target/release/ deno -t deps | sort | uniq | grep -v '#deps'

    ../../buildtools/third_party/libc++/trunk/src/algorithm.cpp
    ../../buildtools/third_party/libc++/trunk/src/any.cpp
    ../../buildtools/third_party/libc++/trunk/src/bind.cpp
    ../../buildtools/third_party/libc++/trunk/src/chrono.cpp
    ../../buildtools/third_party/libc++/trunk/src/condition_variable.cpp
    ../../buildtools/third_party/libc++/trunk/src/debug.cpp
    ../../buildtools/third_party/libc++/trunk/src/exception.cpp
    ../../buildtools/third_party/libc++/trunk/src/functional.cpp
    ../../buildtools/third_party/libc++/trunk/src/future.cpp
    ../../buildtools/third_party/libc++/trunk/src/hash.cpp
    ../../buildtools/third_party/libc++/trunk/src/include/apple_availability.h
    ../../buildtools/third_party/libc++/trunk/src/include/atomic_support.h
    ../../buildtools/third_party/libc++/trunk/src/include/config_elast.h
    ../../buildtools/third_party/libc++/trunk/src/include/refstring.h
    ../../buildtools/third_party/libc++/trunk/src/ios.cpp
    ../../buildtools/third_party/libc++/trunk/src/iostream.cpp
    ../../buildtools/third_party/libc++/trunk/src/locale.cpp
    ../../buildtools/third_party/libc++/trunk/src/memory.cpp
    ../../buildtools/third_party/libc++/trunk/src/mutex.cpp
    ../../buildtools/third_party/libc++/trunk/src/new.cpp
    ../../buildtools/third_party/libc++/trunk/src/optional.cpp
    ../../buildtools/third_party/libc++/trunk/src/random.cpp
    ../../buildtools/third_party/libc++/trunk/src/regex.cpp
    ../../buildtools/third_party/libc++/trunk/src/shared_mutex.cpp
    ../../buildtools/third_party/libc++/trunk/src/stdexcept.cpp
    ../../buildtools/third_party/libc++/trunk/src/string.cpp
    ../../buildtools/third_party/libc++/trunk/src/strstream.cpp
    ../../buildtools/third_party/libc++/trunk/src/support/runtime/exception_libcxxabi.ipp
    ../../buildtools/third_party/libc++/trunk/src/support/runtime/exception_pointer_cxxabi.ipp
    ../../buildtools/third_party/libc++/trunk/src/support/runtime/stdexcept_default.ipp
    ../../buildtools/third_party/libc++/trunk/src/system_error.cpp
    ../../buildtools/third_party/libc++/trunk/src/thread.cpp
    ../../buildtools/third_party/libc++/trunk/src/typeinfo.cpp
    ../../buildtools/third_party/libc++/trunk/src/utility.cpp
    ../../buildtools/third_party/libc++/trunk/src/valarray.cpp
    ../../buildtools/third_party/libc++/trunk/src/variant.cpp
    ../../buildtools/third_party/libc++/trunk/src/vector.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/abort_message.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/abort_message.h
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_aux_runtime.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_default_handlers.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_demangle.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_exception.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_exception.hpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_exception_storage.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_guard.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_handlers.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_handlers.hpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_personality.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_unexpected.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_vector.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/cxa_virtual.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/demangle/DemangleConfig.h
    ../../buildtools/third_party/libc++abi/trunk/src/demangle/ItaniumDemangle.h
    ../../buildtools/third_party/libc++abi/trunk/src/demangle/StringView.h
    ../../buildtools/third_party/libc++abi/trunk/src/demangle/Utility.h
    ../../buildtools/third_party/libc++abi/trunk/src/fallback_malloc.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/fallback_malloc.h
    ../../buildtools/third_party/libc++abi/trunk/src/include/atomic_support.h
    ../../buildtools/third_party/libc++abi/trunk/src/include/refstring.h
    ../../buildtools/third_party/libc++abi/trunk/src/private_typeinfo.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/private_typeinfo.h
    ../../buildtools/third_party/libc++abi/trunk/src/stdlib_exception.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/stdlib_stdexcept.cpp
    ../../buildtools/third_party/libc++abi/trunk/src/stdlib_typeinfo.cpp
    ../../core/libdeno/api.cc
    ../../core/libdeno/binding.cc
    ../../core/libdeno/buffer.h
    ../../core/libdeno/deno.h
    ../../core/libdeno/exceptions.cc
    ../../core/libdeno/exceptions.h
    ../../core/libdeno/file_util.cc
    ../../core/libdeno/file_util.h
    ../../core/libdeno/file_util_test.cc
    ../../core/libdeno/internal.h
    ../../core/libdeno/libdeno_test.cc
    ../../core/libdeno/modules.cc
    ../../core/libdeno/modules_test.cc
    ../../core/libdeno/snapshot_creator.cc
    ../../core/libdeno/test.cc
    ../../core/libdeno/test.h
    ../../testing/gtest/empty.cc
    ../../testing/gtest/include/gtest/gtest.h
    ../../testing/gtest/include/gtest/gtest_prod.h
    ../../third_party/flatbuffers/grpc/src/compiler/config.h
    ../../third_party/flatbuffers/grpc/src/compiler/cpp_generator.cc
    ../../third_party/flatbuffers/grpc/src/compiler/cpp_generator.h
    ../../third_party/flatbuffers/grpc/src/compiler/go_generator.cc
    ../../third_party/flatbuffers/grpc/src/compiler/go_generator.h
    ../../third_party/flatbuffers/grpc/src/compiler/java_generator.cc
    ../../third_party/flatbuffers/grpc/src/compiler/java_generator.h
    ../../third_party/flatbuffers/grpc/src/compiler/schema_interface.h
    ../../third_party/flatbuffers/include/flatbuffers/base.h
    ../../third_party/flatbuffers/include/flatbuffers/code_generators.h
    ../../third_party/flatbuffers/include/flatbuffers/flatbuffers.h
    ../../third_party/flatbuffers/include/flatbuffers/flatc.h
    ../../third_party/flatbuffers/include/flatbuffers/flexbuffers.h
    ../../third_party/flatbuffers/include/flatbuffers/hash.h
    ../../third_party/flatbuffers/include/flatbuffers/idl.h
    ../../third_party/flatbuffers/include/flatbuffers/reflection.h
    ../../third_party/flatbuffers/include/flatbuffers/reflection_generated.h
    ../../third_party/flatbuffers/include/flatbuffers/stl_emulation.h
    ../../third_party/flatbuffers/include/flatbuffers/util.h
    ../../third_party/flatbuffers/src/code_generators.cpp
    ../../third_party/flatbuffers/src/flatc.cpp
    ../../third_party/flatbuffers/src/flatc_main.cpp
    ../../third_party/flatbuffers/src/idl_gen_cpp.cpp
    ../../third_party/flatbuffers/src/idl_gen_dart.cpp
    ../../third_party/flatbuffers/src/idl_gen_fbs.cpp
    ../../third_party/flatbuffers/src/idl_gen_general.cpp
    ../../third_party/flatbuffers/src/idl_gen_go.cpp
    ../../third_party/flatbuffers/src/idl_gen_grpc.cpp
    ../../third_party/flatbuffers/src/idl_gen_js.cpp
    ../../third_party/flatbuffers/src/idl_gen_json_schema.cpp
    ../../third_party/flatbuffers/src/idl_gen_lobster.cpp
    ../../third_party/flatbuffers/src/idl_gen_lua.cpp
    ../../third_party/flatbuffers/src/idl_gen_php.cpp
    ../../third_party/flatbuffers/src/idl_gen_python.cpp
    ../../third_party/flatbuffers/src/idl_gen_rust.cpp
    ../../third_party/flatbuffers/src/idl_gen_text.cpp
    ../../third_party/flatbuffers/src/idl_parser.cpp
    ../../third_party/flatbuffers/src/reflection.cpp
    ../../third_party/flatbuffers/src/util.cpp
    ../../third_party/googletest/custom/gtest/internal/custom/gtest-port.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest-death-test.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest-matchers.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest-message.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest-param-test.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest-printers.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest-spi.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest-test-part.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest-typed-test.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest_pred_impl.h
    ../../third_party/googletest/src/googletest/include/gtest/gtest_prod.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/custom/gtest-printers.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/custom/gtest.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/gtest-death-test-internal.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/gtest-filepath.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/gtest-internal.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/gtest-param-util.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/gtest-port-arch.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/gtest-string.h
    ../../third_party/googletest/src/googletest/include/gtest/internal/gtest-type-util.h
    ../../third_party/googletest/src/googletest/src/gtest-death-test.cc
    ../../third_party/googletest/src/googletest/src/gtest-filepath.cc
    ../../third_party/googletest/src/googletest/src/gtest-internal-inl.h
    ../../third_party/googletest/src/googletest/src/gtest-matchers.cc
    ../../third_party/googletest/src/googletest/src/gtest-port.cc
    ../../third_party/googletest/src/googletest/src/gtest-printers.cc
    ../../third_party/googletest/src/googletest/src/gtest-test-part.cc
    ../../third_party/googletest/src/googletest/src/gtest-typed-test.cc
    ../../third_party/googletest/src/googletest/src/gtest.cc
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/block.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/block.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/cpu-intel.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/crypto.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/bn/generic.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/bn/internal.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/bn/montgomery.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/bn/montgomery_inv.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz256.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz256.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz256_table.inl
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz384.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/ecp_nistz384.inl
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/gfp_p256.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/ec/gfp_p384.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/modes/gcm.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/fipsmodule/modes/internal.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/internal.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/limbs/limbs.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/limbs/limbs.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/limbs/limbs.inl
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/crypto/mem.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/include/GFp/base.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/include/GFp/cpu.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/include/GFp/mem.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/include/GFp/type_check.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/aes-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/aesni-gcm-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/aesni-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/chacha-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/ghash-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/p256-x86_64-asm-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/poly1305-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/sha256-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/sha512-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/vpaes-x86_64-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/x86_64-mont-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/pregenerated/x86_64-mont5-macosx.S
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/third_party/fiat/curve25519.c
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/third_party/fiat/curve25519_tables.h
    ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/ring-0.14.6/third_party/fiat/internal.h
    ../../third_party/v8/base/trace_event/common/trace_event_common.h
    ../../third_party/v8/include/libplatform/libplatform-export.h
    ../../third_party/v8/include/libplatform/libplatform.h
    ../../third_party/v8/include/libplatform/v8-tracing.h
    ../../third_party/v8/include/v8-inspector.h
    ../../third_party/v8/include/v8-internal.h
    ../../third_party/v8/include/v8-platform.h
    ../../third_party/v8/include/v8-profiler.h
    ../../third_party/v8/include/v8-testing.h
    ../../third_party/v8/include/v8-util.h
    ../../third_party/v8/include/v8-value-serializer-version.h
    ../../third_party/v8/include/v8-version-string.h
    ../../third_party/v8/include/v8-version.h
    ../../third_party/v8/include/v8-wasm-trap-handler-posix.h
    ../../third_party/v8/include/v8.h
    ../../third_party/v8/include/v8config.h
    ../../third_party/v8/src/accessors.cc
    ../../third_party/v8/src/accessors.h
    ../../third_party/v8/src/address-map.cc
    ../../third_party/v8/src/address-map.h
    ../../third_party/v8/src/allocation-site-scopes-inl.h
    ../../third_party/v8/src/allocation-site-scopes.h
    ../../third_party/v8/src/allocation.cc
    ../../third_party/v8/src/allocation.h
    ../../third_party/v8/src/api-arguments-inl.h
    ../../third_party/v8/src/api-arguments.cc
    ../../third_party/v8/src/api-arguments.h
    ../../third_party/v8/src/api-inl.h
    ../../third_party/v8/src/api-natives.cc
    ../../third_party/v8/src/api-natives.h
    ../../third_party/v8/src/api.cc
    ../../third_party/v8/src/api.h
    ../../third_party/v8/src/arguments-inl.h
    ../../third_party/v8/src/arguments.cc
    ../../third_party/v8/src/arguments.h
    ../../third_party/v8/src/asan.h
    ../../third_party/v8/src/asmjs/asm-js.cc
    ../../third_party/v8/src/asmjs/asm-js.h
    ../../third_party/v8/src/asmjs/asm-names.h
    ../../third_party/v8/src/asmjs/asm-parser.cc
    ../../third_party/v8/src/asmjs/asm-parser.h
    ../../third_party/v8/src/asmjs/asm-scanner.cc
    ../../third_party/v8/src/asmjs/asm-scanner.h
    ../../third_party/v8/src/asmjs/asm-types.cc
    ../../third_party/v8/src/asmjs/asm-types.h
    ../../third_party/v8/src/assembler-arch.h
    ../../third_party/v8/src/assembler-inl.h
    ../../third_party/v8/src/assembler.cc
    ../../third_party/v8/src/assembler.h
    ../../third_party/v8/src/assert-scope.cc
    ../../third_party/v8/src/assert-scope.h
    ../../third_party/v8/src/ast/ast-function-literal-id-reindexer.cc
    ../../third_party/v8/src/ast/ast-function-literal-id-reindexer.h
    ../../third_party/v8/src/ast/ast-source-ranges.h
    ../../third_party/v8/src/ast/ast-traversal-visitor.h
    ../../third_party/v8/src/ast/ast-value-factory.cc
    ../../third_party/v8/src/ast/ast-value-factory.h
    ../../third_party/v8/src/ast/ast.cc
    ../../third_party/v8/src/ast/ast.h
    ../../third_party/v8/src/ast/modules.cc
    ../../third_party/v8/src/ast/modules.h
    ../../third_party/v8/src/ast/prettyprinter.cc
    ../../third_party/v8/src/ast/prettyprinter.h
    ../../third_party/v8/src/ast/scopes.cc
    ../../third_party/v8/src/ast/scopes.h
    ../../third_party/v8/src/ast/source-range-ast-visitor.cc
    ../../third_party/v8/src/ast/source-range-ast-visitor.h
    ../../third_party/v8/src/ast/variables.cc
    ../../third_party/v8/src/ast/variables.h
    ../../third_party/v8/src/bailout-reason.cc
    ../../third_party/v8/src/bailout-reason.h
    ../../third_party/v8/src/base/adapters.h
    ../../third_party/v8/src/base/address-region.h
    ../../third_party/v8/src/base/atomic-utils.h
    ../../third_party/v8/src/base/atomicops.h
    ../../third_party/v8/src/base/atomicops_internals_atomicword_compat.h
    ../../third_party/v8/src/base/atomicops_internals_portable.h
    ../../third_party/v8/src/base/base-export.h
    ../../third_party/v8/src/base/bits.cc
    ../../third_party/v8/src/base/bits.h
    ../../third_party/v8/src/base/bounded-page-allocator.cc
    ../../third_party/v8/src/base/bounded-page-allocator.h
    ../../third_party/v8/src/base/build_config.h
    ../../third_party/v8/src/base/compiler-specific.h
    ../../third_party/v8/src/base/cpu.cc
    ../../third_party/v8/src/base/cpu.h
    ../../third_party/v8/src/base/debug/stack_trace.cc
    ../../third_party/v8/src/base/debug/stack_trace.h
    ../../third_party/v8/src/base/debug/stack_trace_posix.cc
    ../../third_party/v8/src/base/division-by-constant.cc
    ../../third_party/v8/src/base/division-by-constant.h
    ../../third_party/v8/src/base/enum-set.h
    ../../third_party/v8/src/base/export-template.h
    ../../third_party/v8/src/base/file-utils.cc
    ../../third_party/v8/src/base/file-utils.h
    ../../third_party/v8/src/base/flags.h
    ../../third_party/v8/src/base/format-macros.h
    ../../third_party/v8/src/base/free_deleter.h
    ../../third_party/v8/src/base/functional.cc
    ../../third_party/v8/src/base/functional.h
    ../../third_party/v8/src/base/hashmap-entry.h
    ../../third_party/v8/src/base/hashmap.h
    ../../third_party/v8/src/base/ieee754.cc
    ../../third_party/v8/src/base/ieee754.h
    ../../third_party/v8/src/base/iterator.h
    ../../third_party/v8/src/base/lazy-instance.h
    ../../third_party/v8/src/base/list.h
    ../../third_party/v8/src/base/logging.cc
    ../../third_party/v8/src/base/logging.h
    ../../third_party/v8/src/base/lsan-page-allocator.h
    ../../third_party/v8/src/base/macros.h
    ../../third_party/v8/src/base/once.cc
    ../../third_party/v8/src/base/once.h
    ../../third_party/v8/src/base/optional.h
    ../../third_party/v8/src/base/overflowing-math.h
    ../../third_party/v8/src/base/page-allocator.cc
    ../../third_party/v8/src/base/page-allocator.h
    ../../third_party/v8/src/base/platform/condition-variable.cc
    ../../third_party/v8/src/base/platform/condition-variable.h
    ../../third_party/v8/src/base/platform/elapsed-timer.h
    ../../third_party/v8/src/base/platform/mutex.cc
    ../../third_party/v8/src/base/platform/mutex.h
    ../../third_party/v8/src/base/platform/platform-macos.cc
    ../../third_party/v8/src/base/platform/platform-posix-time.cc
    ../../third_party/v8/src/base/platform/platform-posix-time.h
    ../../third_party/v8/src/base/platform/platform-posix.cc
    ../../third_party/v8/src/base/platform/platform-posix.h
    ../../third_party/v8/src/base/platform/platform.h
    ../../third_party/v8/src/base/platform/semaphore.cc
    ../../third_party/v8/src/base/platform/semaphore.h
    ../../third_party/v8/src/base/platform/time.cc
    ../../third_party/v8/src/base/platform/time.h
    ../../third_party/v8/src/base/region-allocator.cc
    ../../third_party/v8/src/base/region-allocator.h
    ../../third_party/v8/src/base/ring-buffer.h
    ../../third_party/v8/src/base/safe_conversions.h
    ../../third_party/v8/src/base/safe_conversions_impl.h
    ../../third_party/v8/src/base/small-vector.h
    ../../third_party/v8/src/base/sys-info.cc
    ../../third_party/v8/src/base/sys-info.h
    ../../third_party/v8/src/base/template-utils.h
    ../../third_party/v8/src/base/threaded-list.h
    ../../third_party/v8/src/base/timezone-cache.h
    ../../third_party/v8/src/base/tsan.h
    ../../third_party/v8/src/base/utils/random-number-generator.cc
    ../../third_party/v8/src/base/utils/random-number-generator.h
    ../../third_party/v8/src/base/v8-fallthrough.h
    ../../third_party/v8/src/basic-block-profiler.cc
    ../../third_party/v8/src/basic-block-profiler.h
    ../../third_party/v8/src/bignum-dtoa.cc
    ../../third_party/v8/src/bignum-dtoa.h
    ../../third_party/v8/src/bignum.cc
    ../../third_party/v8/src/bignum.h
    ../../third_party/v8/src/bit-vector.cc
    ../../third_party/v8/src/bit-vector.h
    ../../third_party/v8/src/bootstrapper.cc
    ../../third_party/v8/src/bootstrapper.h
    ../../third_party/v8/src/boxed-float.h
    ../../third_party/v8/src/builtins/accessors.h
    ../../third_party/v8/src/builtins/builtins-api.cc
    ../../third_party/v8/src/builtins/builtins-arguments-gen.cc
    ../../third_party/v8/src/builtins/builtins-arguments-gen.h
    ../../third_party/v8/src/builtins/builtins-array-gen.cc
    ../../third_party/v8/src/builtins/builtins-array-gen.h
    ../../third_party/v8/src/builtins/builtins-array.cc
    ../../third_party/v8/src/builtins/builtins-arraybuffer.cc
    ../../third_party/v8/src/builtins/builtins-async-function-gen.cc
    ../../third_party/v8/src/builtins/builtins-async-gen.cc
    ../../third_party/v8/src/builtins/builtins-async-gen.h
    ../../third_party/v8/src/builtins/builtins-async-generator-gen.cc
    ../../third_party/v8/src/builtins/builtins-async-iterator-gen.cc
    ../../third_party/v8/src/builtins/builtins-bigint-gen.cc
    ../../third_party/v8/src/builtins/builtins-bigint.cc
    ../../third_party/v8/src/builtins/builtins-boolean-gen.cc
    ../../third_party/v8/src/builtins/builtins-boolean.cc
    ../../third_party/v8/src/builtins/builtins-call-gen.cc
    ../../third_party/v8/src/builtins/builtins-call-gen.h
    ../../third_party/v8/src/builtins/builtins-call.cc
    ../../third_party/v8/src/builtins/builtins-callsite.cc
    ../../third_party/v8/src/builtins/builtins-collections-gen.cc
    ../../third_party/v8/src/builtins/builtins-collections-gen.h
    ../../third_party/v8/src/builtins/builtins-collections.cc
    ../../third_party/v8/src/builtins/builtins-console-gen.cc
    ../../third_party/v8/src/builtins/builtins-console.cc
    ../../third_party/v8/src/builtins/builtins-constructor-gen.cc
    ../../third_party/v8/src/builtins/builtins-constructor-gen.h
    ../../third_party/v8/src/builtins/builtins-constructor.h
    ../../third_party/v8/src/builtins/builtins-conversion-gen.cc
    ../../third_party/v8/src/builtins/builtins-data-view-gen.h
    ../../third_party/v8/src/builtins/builtins-dataview.cc
    ../../third_party/v8/src/builtins/builtins-date-gen.cc
    ../../third_party/v8/src/builtins/builtins-date.cc
    ../../third_party/v8/src/builtins/builtins-debug-gen.cc
    ../../third_party/v8/src/builtins/builtins-definitions.h
    ../../third_party/v8/src/builtins/builtins-descriptors.h
    ../../third_party/v8/src/builtins/builtins-error.cc
    ../../third_party/v8/src/builtins/builtins-extras-utils.cc
    ../../third_party/v8/src/builtins/builtins-function-gen.cc
    ../../third_party/v8/src/builtins/builtins-function.cc
    ../../third_party/v8/src/builtins/builtins-generator-gen.cc
    ../../third_party/v8/src/builtins/builtins-global-gen.cc
    ../../third_party/v8/src/builtins/builtins-global.cc
    ../../third_party/v8/src/builtins/builtins-handler-gen.cc
    ../../third_party/v8/src/builtins/builtins-ic-gen.cc
    ../../third_party/v8/src/builtins/builtins-internal-gen.cc
    ../../third_party/v8/src/builtins/builtins-internal.cc
    ../../third_party/v8/src/builtins/builtins-interpreter-gen.cc
    ../../third_party/v8/src/builtins/builtins-iterator-gen.cc
    ../../third_party/v8/src/builtins/builtins-iterator-gen.h
    ../../third_party/v8/src/builtins/builtins-json.cc
    ../../third_party/v8/src/builtins/builtins-lazy-gen.cc
    ../../third_party/v8/src/builtins/builtins-lazy-gen.h
    ../../third_party/v8/src/builtins/builtins-math-gen.cc
    ../../third_party/v8/src/builtins/builtins-math-gen.h
    ../../third_party/v8/src/builtins/builtins-math.cc
    ../../third_party/v8/src/builtins/builtins-microtask-queue-gen.cc
    ../../third_party/v8/src/builtins/builtins-number-gen.cc
    ../../third_party/v8/src/builtins/builtins-number.cc
    ../../third_party/v8/src/builtins/builtins-object-gen.cc
    ../../third_party/v8/src/builtins/builtins-object-gen.h
    ../../third_party/v8/src/builtins/builtins-object.cc
    ../../third_party/v8/src/builtins/builtins-promise-gen.cc
    ../../third_party/v8/src/builtins/builtins-promise-gen.h
    ../../third_party/v8/src/builtins/builtins-promise.cc
    ../../third_party/v8/src/builtins/builtins-promise.h
    ../../third_party/v8/src/builtins/builtins-proxy-gen.cc
    ../../third_party/v8/src/builtins/builtins-proxy-gen.h
    ../../third_party/v8/src/builtins/builtins-reflect-gen.cc
    ../../third_party/v8/src/builtins/builtins-reflect.cc
    ../../third_party/v8/src/builtins/builtins-regexp-gen.cc
    ../../third_party/v8/src/builtins/builtins-regexp-gen.h
    ../../third_party/v8/src/builtins/builtins-regexp.cc
    ../../third_party/v8/src/builtins/builtins-sharedarraybuffer-gen.cc
    ../../third_party/v8/src/builtins/builtins-sharedarraybuffer.cc
    ../../third_party/v8/src/builtins/builtins-string-gen.cc
    ../../third_party/v8/src/builtins/builtins-string-gen.h
    ../../third_party/v8/src/builtins/builtins-string.cc
    ../../third_party/v8/src/builtins/builtins-symbol-gen.cc
    ../../third_party/v8/src/builtins/builtins-symbol.cc
    ../../third_party/v8/src/builtins/builtins-trace.cc
    ../../third_party/v8/src/builtins/builtins-typed-array-gen.cc
    ../../third_party/v8/src/builtins/builtins-typed-array-gen.h
    ../../third_party/v8/src/builtins/builtins-typed-array.cc
    ../../third_party/v8/src/builtins/builtins-utils-gen.h
    ../../third_party/v8/src/builtins/builtins-utils-inl.h
    ../../third_party/v8/src/builtins/builtins-utils.h
    ../../third_party/v8/src/builtins/builtins-wasm-gen.cc
    ../../third_party/v8/src/builtins/builtins-weak-refs.cc
    ../../third_party/v8/src/builtins/builtins.cc
    ../../third_party/v8/src/builtins/builtins.h
    ../../third_party/v8/src/builtins/constants-table-builder.cc
    ../../third_party/v8/src/builtins/constants-table-builder.h
    ../../third_party/v8/src/builtins/generate-bytecodes-builtins-list.cc
    ../../third_party/v8/src/builtins/growable-fixed-array-gen.cc
    ../../third_party/v8/src/builtins/growable-fixed-array-gen.h
    ../../third_party/v8/src/builtins/setup-builtins-internal.cc
    ../../third_party/v8/src/builtins/x64/builtins-x64.cc
    ../../third_party/v8/src/cached-powers.cc
    ../../third_party/v8/src/cached-powers.h
    ../../third_party/v8/src/callable.h
    ../../third_party/v8/src/cancelable-task.cc
    ../../third_party/v8/src/cancelable-task.h
    ../../third_party/v8/src/char-predicates-inl.h
    ../../third_party/v8/src/char-predicates.h
    ../../third_party/v8/src/checks.h
    ../../third_party/v8/src/code-comments.cc
    ../../third_party/v8/src/code-comments.h
    ../../third_party/v8/src/code-desc.cc
    ../../third_party/v8/src/code-desc.h
    ../../third_party/v8/src/code-events.h
    ../../third_party/v8/src/code-factory.cc
    ../../third_party/v8/src/code-factory.h
    ../../third_party/v8/src/code-reference.cc
    ../../third_party/v8/src/code-reference.h
    ../../third_party/v8/src/code-stub-assembler.cc
    ../../third_party/v8/src/code-stub-assembler.h
    ../../third_party/v8/src/code-tracer.h
    ../../third_party/v8/src/codegen/assembler-arch.h
    ../../third_party/v8/src/codegen/assembler.h
    ../../third_party/v8/src/codegen/bailout-reason.h
    ../../third_party/v8/src/codegen/code-comments.h
    ../../third_party/v8/src/codegen/code-desc.h
    ../../third_party/v8/src/codegen/constants-arch.h
    ../../third_party/v8/src/codegen/cpu-features.h
    ../../third_party/v8/src/codegen/external-reference-table.h
    ../../third_party/v8/src/codegen/external-reference.h
    ../../third_party/v8/src/codegen/flush-instruction-cache.h
    ../../third_party/v8/src/codegen/handler-table.h
    ../../third_party/v8/src/codegen/label.h
    ../../third_party/v8/src/codegen/machine-type.h
    ../../third_party/v8/src/codegen/register-arch.h
    ../../third_party/v8/src/codegen/register.h
    ../../third_party/v8/src/codegen/reglist.h
    ../../third_party/v8/src/codegen/reloc-info.h
    ../../third_party/v8/src/codegen/safepoint-table.h
    ../../third_party/v8/src/codegen/signature.h
    ../../third_party/v8/src/codegen/source-position-table.h
    ../../third_party/v8/src/codegen/source-position.h
    ../../third_party/v8/src/codegen/x64/assembler-x64.h
    ../../third_party/v8/src/codegen/x64/constants-x64.h
    ../../third_party/v8/src/codegen/x64/register-x64.h
    ../../third_party/v8/src/codegen/x64/sse-instr.h
    ../../third_party/v8/src/common/assert-scope.h
    ../../third_party/v8/src/common/checks.h
    ../../third_party/v8/src/common/globals.h
    ../../third_party/v8/src/common/ptr-compr-inl.h
    ../../third_party/v8/src/common/ptr-compr.h
    ../../third_party/v8/src/common/v8memory.h
    ../../third_party/v8/src/compilation-cache.cc
    ../../third_party/v8/src/compilation-cache.h
    ../../third_party/v8/src/compilation-statistics.cc
    ../../third_party/v8/src/compilation-statistics.h
    ../../third_party/v8/src/compiler-dispatcher/compiler-dispatcher.cc
    ../../third_party/v8/src/compiler-dispatcher/compiler-dispatcher.h
    ../../third_party/v8/src/compiler-dispatcher/optimizing-compile-dispatcher.cc
    ../../third_party/v8/src/compiler-dispatcher/optimizing-compile-dispatcher.h
    ../../third_party/v8/src/compiler.cc
    ../../third_party/v8/src/compiler.h
    ../../third_party/v8/src/compiler/access-builder.cc
    ../../third_party/v8/src/compiler/access-builder.h
    ../../third_party/v8/src/compiler/access-info.cc
    ../../third_party/v8/src/compiler/access-info.h
    ../../third_party/v8/src/compiler/all-nodes.cc
    ../../third_party/v8/src/compiler/all-nodes.h
    ../../third_party/v8/src/compiler/allocation-builder-inl.h
    ../../third_party/v8/src/compiler/allocation-builder.h
    ../../third_party/v8/src/compiler/backend/code-generator-impl.h
    ../../third_party/v8/src/compiler/backend/code-generator.cc
    ../../third_party/v8/src/compiler/backend/code-generator.h
    ../../third_party/v8/src/compiler/backend/frame-elider.cc
    ../../third_party/v8/src/compiler/backend/frame-elider.h
    ../../third_party/v8/src/compiler/backend/gap-resolver.cc
    ../../third_party/v8/src/compiler/backend/gap-resolver.h
    ../../third_party/v8/src/compiler/backend/instruction-codes.h
    ../../third_party/v8/src/compiler/backend/instruction-scheduler.cc
    ../../third_party/v8/src/compiler/backend/instruction-scheduler.h
    ../../third_party/v8/src/compiler/backend/instruction-selector-impl.h
    ../../third_party/v8/src/compiler/backend/instruction-selector.cc
    ../../third_party/v8/src/compiler/backend/instruction-selector.h
    ../../third_party/v8/src/compiler/backend/instruction.cc
    ../../third_party/v8/src/compiler/backend/instruction.h
    ../../third_party/v8/src/compiler/backend/jump-threading.cc
    ../../third_party/v8/src/compiler/backend/jump-threading.h
    ../../third_party/v8/src/compiler/backend/live-range-separator.cc
    ../../third_party/v8/src/compiler/backend/live-range-separator.h
    ../../third_party/v8/src/compiler/backend/move-optimizer.cc
    ../../third_party/v8/src/compiler/backend/move-optimizer.h
    ../../third_party/v8/src/compiler/backend/register-allocator-verifier.cc
    ../../third_party/v8/src/compiler/backend/register-allocator-verifier.h
    ../../third_party/v8/src/compiler/backend/register-allocator.cc
    ../../third_party/v8/src/compiler/backend/register-allocator.h
    ../../third_party/v8/src/compiler/backend/unwinding-info-writer.h
    ../../third_party/v8/src/compiler/backend/x64/code-generator-x64.cc
    ../../third_party/v8/src/compiler/backend/x64/instruction-codes-x64.h
    ../../third_party/v8/src/compiler/backend/x64/instruction-scheduler-x64.cc
    ../../third_party/v8/src/compiler/backend/x64/instruction-selector-x64.cc
    ../../third_party/v8/src/compiler/backend/x64/unwinding-info-writer-x64.cc
    ../../third_party/v8/src/compiler/backend/x64/unwinding-info-writer-x64.h
    ../../third_party/v8/src/compiler/basic-block-instrumentor.cc
    ../../third_party/v8/src/compiler/basic-block-instrumentor.h
    ../../third_party/v8/src/compiler/branch-elimination.cc
    ../../third_party/v8/src/compiler/branch-elimination.h
    ../../third_party/v8/src/compiler/bytecode-analysis.cc
    ../../third_party/v8/src/compiler/bytecode-analysis.h
    ../../third_party/v8/src/compiler/bytecode-graph-builder.cc
    ../../third_party/v8/src/compiler/bytecode-graph-builder.h
    ../../third_party/v8/src/compiler/bytecode-liveness-map.cc
    ../../third_party/v8/src/compiler/bytecode-liveness-map.h
    ../../third_party/v8/src/compiler/c-linkage.cc
    ../../third_party/v8/src/compiler/checkpoint-elimination.cc
    ../../third_party/v8/src/compiler/checkpoint-elimination.h
    ../../third_party/v8/src/compiler/code-assembler.cc
    ../../third_party/v8/src/compiler/code-assembler.h
    ../../third_party/v8/src/compiler/common-node-cache.cc
    ../../third_party/v8/src/compiler/common-node-cache.h
    ../../third_party/v8/src/compiler/common-operator-reducer.cc
    ../../third_party/v8/src/compiler/common-operator-reducer.h
    ../../third_party/v8/src/compiler/common-operator.cc
    ../../third_party/v8/src/compiler/common-operator.h
    ../../third_party/v8/src/compiler/compilation-dependencies.cc
    ../../third_party/v8/src/compiler/compilation-dependencies.h
    ../../third_party/v8/src/compiler/compiler-source-position-table.cc
    ../../third_party/v8/src/compiler/compiler-source-position-table.h
    ../../third_party/v8/src/compiler/constant-folding-reducer.cc
    ../../third_party/v8/src/compiler/constant-folding-reducer.h
    ../../third_party/v8/src/compiler/control-equivalence.cc
    ../../third_party/v8/src/compiler/control-equivalence.h
    ../../third_party/v8/src/compiler/control-flow-optimizer.cc
    ../../third_party/v8/src/compiler/control-flow-optimizer.h
    ../../third_party/v8/src/compiler/dead-code-elimination.cc
    ../../third_party/v8/src/compiler/dead-code-elimination.h
    ../../third_party/v8/src/compiler/diamond.h
    ../../third_party/v8/src/compiler/effect-control-linearizer.cc
    ../../third_party/v8/src/compiler/effect-control-linearizer.h
    ../../third_party/v8/src/compiler/escape-analysis-reducer.cc
    ../../third_party/v8/src/compiler/escape-analysis-reducer.h
    ../../third_party/v8/src/compiler/escape-analysis.cc
    ../../third_party/v8/src/compiler/escape-analysis.h
    ../../third_party/v8/src/compiler/frame-states.cc
    ../../third_party/v8/src/compiler/frame-states.h
    ../../third_party/v8/src/compiler/frame.cc
    ../../third_party/v8/src/compiler/frame.h
    ../../third_party/v8/src/compiler/functional-list.h
    ../../third_party/v8/src/compiler/graph-assembler.cc
    ../../third_party/v8/src/compiler/graph-assembler.h
    ../../third_party/v8/src/compiler/graph-reducer.cc
    ../../third_party/v8/src/compiler/graph-reducer.h
    ../../third_party/v8/src/compiler/graph-trimmer.cc
    ../../third_party/v8/src/compiler/graph-trimmer.h
    ../../third_party/v8/src/compiler/graph-visualizer.cc
    ../../third_party/v8/src/compiler/graph-visualizer.h
    ../../third_party/v8/src/compiler/graph.cc
    ../../third_party/v8/src/compiler/graph.h
    ../../third_party/v8/src/compiler/int64-lowering.cc
    ../../third_party/v8/src/compiler/int64-lowering.h
    ../../third_party/v8/src/compiler/js-call-reducer.cc
    ../../third_party/v8/src/compiler/js-call-reducer.h
    ../../third_party/v8/src/compiler/js-context-specialization.cc
    ../../third_party/v8/src/compiler/js-context-specialization.h
    ../../third_party/v8/src/compiler/js-create-lowering.cc
    ../../third_party/v8/src/compiler/js-create-lowering.h
    ../../third_party/v8/src/compiler/js-generic-lowering.cc
    ../../third_party/v8/src/compiler/js-generic-lowering.h
    ../../third_party/v8/src/compiler/js-graph.cc
    ../../third_party/v8/src/compiler/js-graph.h
    ../../third_party/v8/src/compiler/js-heap-broker.cc
    ../../third_party/v8/src/compiler/js-heap-broker.h
    ../../third_party/v8/src/compiler/js-heap-copy-reducer.cc
    ../../third_party/v8/src/compiler/js-heap-copy-reducer.h
    ../../third_party/v8/src/compiler/js-inlining-heuristic.cc
    ../../third_party/v8/src/compiler/js-inlining-heuristic.h
    ../../third_party/v8/src/compiler/js-inlining.cc
    ../../third_party/v8/src/compiler/js-inlining.h
    ../../third_party/v8/src/compiler/js-intrinsic-lowering.cc
    ../../third_party/v8/src/compiler/js-intrinsic-lowering.h
    ../../third_party/v8/src/compiler/js-native-context-specialization.cc
    ../../third_party/v8/src/compiler/js-native-context-specialization.h
    ../../third_party/v8/src/compiler/js-operator.cc
    ../../third_party/v8/src/compiler/js-operator.h
    ../../third_party/v8/src/compiler/js-type-hint-lowering.cc
    ../../third_party/v8/src/compiler/js-type-hint-lowering.h
    ../../third_party/v8/src/compiler/js-typed-lowering.cc
    ../../third_party/v8/src/compiler/js-typed-lowering.h
    ../../third_party/v8/src/compiler/linkage.cc
    ../../third_party/v8/src/compiler/linkage.h
    ../../third_party/v8/src/compiler/load-elimination.cc
    ../../third_party/v8/src/compiler/load-elimination.h
    ../../third_party/v8/src/compiler/loop-analysis.cc
    ../../third_party/v8/src/compiler/loop-analysis.h
    ../../third_party/v8/src/compiler/loop-peeling.cc
    ../../third_party/v8/src/compiler/loop-peeling.h
    ../../third_party/v8/src/compiler/loop-variable-optimizer.cc
    ../../third_party/v8/src/compiler/loop-variable-optimizer.h
    ../../third_party/v8/src/compiler/machine-graph-verifier.cc
    ../../third_party/v8/src/compiler/machine-graph-verifier.h
    ../../third_party/v8/src/compiler/machine-graph.cc
    ../../third_party/v8/src/compiler/machine-graph.h
    ../../third_party/v8/src/compiler/machine-operator-reducer.cc
    ../../third_party/v8/src/compiler/machine-operator-reducer.h
    ../../third_party/v8/src/compiler/machine-operator.cc
    ../../third_party/v8/src/compiler/machine-operator.h
    ../../third_party/v8/src/compiler/memory-optimizer.cc
    ../../third_party/v8/src/compiler/memory-optimizer.h
    ../../third_party/v8/src/compiler/node-aux-data.h
    ../../third_party/v8/src/compiler/node-cache.cc
    ../../third_party/v8/src/compiler/node-cache.h
    ../../third_party/v8/src/compiler/node-marker.cc
    ../../third_party/v8/src/compiler/node-marker.h
    ../../third_party/v8/src/compiler/node-matchers.cc
    ../../third_party/v8/src/compiler/node-matchers.h
    ../../third_party/v8/src/compiler/node-origin-table.cc
    ../../third_party/v8/src/compiler/node-origin-table.h
    ../../third_party/v8/src/compiler/node-properties.cc
    ../../third_party/v8/src/compiler/node-properties.h
    ../../third_party/v8/src/compiler/node.cc
    ../../third_party/v8/src/compiler/node.h
    ../../third_party/v8/src/compiler/opcodes.cc
    ../../third_party/v8/src/compiler/opcodes.h
    ../../third_party/v8/src/compiler/operation-typer.cc
    ../../third_party/v8/src/compiler/operation-typer.h
    ../../third_party/v8/src/compiler/operator-properties.cc
    ../../third_party/v8/src/compiler/operator-properties.h
    ../../third_party/v8/src/compiler/operator.cc
    ../../third_party/v8/src/compiler/operator.h
    ../../third_party/v8/src/compiler/osr.cc
    ../../third_party/v8/src/compiler/osr.h
    ../../third_party/v8/src/compiler/per-isolate-compiler-cache.h
    ../../third_party/v8/src/compiler/persistent-map.h
    ../../third_party/v8/src/compiler/pipeline-statistics.cc
    ../../third_party/v8/src/compiler/pipeline-statistics.h
    ../../third_party/v8/src/compiler/pipeline.cc
    ../../third_party/v8/src/compiler/pipeline.h
    ../../third_party/v8/src/compiler/property-access-builder.cc
    ../../third_party/v8/src/compiler/property-access-builder.h
    ../../third_party/v8/src/compiler/raw-machine-assembler.cc
    ../../third_party/v8/src/compiler/raw-machine-assembler.h
    ../../third_party/v8/src/compiler/redundancy-elimination.cc
    ../../third_party/v8/src/compiler/redundancy-elimination.h
    ../../third_party/v8/src/compiler/refs-map.cc
    ../../third_party/v8/src/compiler/refs-map.h
    ../../third_party/v8/src/compiler/representation-change.cc
    ../../third_party/v8/src/compiler/representation-change.h
    ../../third_party/v8/src/compiler/schedule.cc
    ../../third_party/v8/src/compiler/schedule.h
    ../../third_party/v8/src/compiler/scheduler.cc
    ../../third_party/v8/src/compiler/scheduler.h
    ../../third_party/v8/src/compiler/select-lowering.cc
    ../../third_party/v8/src/compiler/select-lowering.h
    ../../third_party/v8/src/compiler/serializer-for-background-compilation.cc
    ../../third_party/v8/src/compiler/serializer-for-background-compilation.h
    ../../third_party/v8/src/compiler/simd-scalar-lowering.cc
    ../../third_party/v8/src/compiler/simd-scalar-lowering.h
    ../../third_party/v8/src/compiler/simplified-lowering.cc
    ../../third_party/v8/src/compiler/simplified-lowering.h
    ../../third_party/v8/src/compiler/simplified-operator-reducer.cc
    ../../third_party/v8/src/compiler/simplified-operator-reducer.h
    ../../third_party/v8/src/compiler/simplified-operator.cc
    ../../third_party/v8/src/compiler/simplified-operator.h
    ../../third_party/v8/src/compiler/state-values-utils.cc
    ../../third_party/v8/src/compiler/state-values-utils.h
    ../../third_party/v8/src/compiler/store-store-elimination.cc
    ../../third_party/v8/src/compiler/store-store-elimination.h
    ../../third_party/v8/src/compiler/type-cache.cc
    ../../third_party/v8/src/compiler/type-cache.h
    ../../third_party/v8/src/compiler/type-narrowing-reducer.cc
    ../../third_party/v8/src/compiler/type-narrowing-reducer.h
    ../../third_party/v8/src/compiler/typed-optimization.cc
    ../../third_party/v8/src/compiler/typed-optimization.h
    ../../third_party/v8/src/compiler/typer.cc
    ../../third_party/v8/src/compiler/typer.h
    ../../third_party/v8/src/compiler/types.cc
    ../../third_party/v8/src/compiler/types.h
    ../../third_party/v8/src/compiler/value-numbering-reducer.cc
    ../../third_party/v8/src/compiler/value-numbering-reducer.h
    ../../third_party/v8/src/compiler/verifier.cc
    ../../third_party/v8/src/compiler/verifier.h
    ../../third_party/v8/src/compiler/wasm-compiler.cc
    ../../third_party/v8/src/compiler/wasm-compiler.h
    ../../third_party/v8/src/compiler/zone-stats.cc
    ../../third_party/v8/src/compiler/zone-stats.h
    ../../third_party/v8/src/constant-pool.cc
    ../../third_party/v8/src/constant-pool.h
    ../../third_party/v8/src/constants-arch.h
    ../../third_party/v8/src/contexts-inl.h
    ../../third_party/v8/src/contexts.cc
    ../../third_party/v8/src/contexts.h
    ../../third_party/v8/src/conversions-inl.h
    ../../third_party/v8/src/conversions.cc
    ../../third_party/v8/src/conversions.h
    ../../third_party/v8/src/counters-definitions.h
    ../../third_party/v8/src/counters-inl.h
    ../../third_party/v8/src/counters.cc
    ../../third_party/v8/src/counters.h
    ../../third_party/v8/src/cpu-features.h
    ../../third_party/v8/src/date.cc
    ../../third_party/v8/src/date.h
    ../../third_party/v8/src/dateparser-inl.h
    ../../third_party/v8/src/dateparser.cc
    ../../third_party/v8/src/dateparser.h
    ../../third_party/v8/src/debug/debug-coverage.cc
    ../../third_party/v8/src/debug/debug-coverage.h
    ../../third_party/v8/src/debug/debug-evaluate.cc
    ../../third_party/v8/src/debug/debug-evaluate.h
    ../../third_party/v8/src/debug/debug-frames.cc
    ../../third_party/v8/src/debug/debug-frames.h
    ../../third_party/v8/src/debug/debug-interface.h
    ../../third_party/v8/src/debug/debug-property-iterator.cc
    ../../third_party/v8/src/debug/debug-property-iterator.h
    ../../third_party/v8/src/debug/debug-scope-iterator.cc
    ../../third_party/v8/src/debug/debug-scope-iterator.h
    ../../third_party/v8/src/debug/debug-scopes.cc
    ../../third_party/v8/src/debug/debug-scopes.h
    ../../third_party/v8/src/debug/debug-stack-trace-iterator.cc
    ../../third_party/v8/src/debug/debug-stack-trace-iterator.h
    ../../third_party/v8/src/debug/debug-type-profile.cc
    ../../third_party/v8/src/debug/debug-type-profile.h
    ../../third_party/v8/src/debug/debug.cc
    ../../third_party/v8/src/debug/debug.h
    ../../third_party/v8/src/debug/interface-types.h
    ../../third_party/v8/src/debug/liveedit.cc
    ../../third_party/v8/src/debug/liveedit.h
    ../../third_party/v8/src/debug/x64/debug-x64.cc
    ../../third_party/v8/src/deoptimize-reason.cc
    ../../third_party/v8/src/deoptimize-reason.h
    ../../third_party/v8/src/deoptimizer.cc
    ../../third_party/v8/src/deoptimizer.h
    ../../third_party/v8/src/deoptimizer/deoptimize-reason.h
    ../../third_party/v8/src/deoptimizer/deoptimizer.h
    ../../third_party/v8/src/detachable-vector.cc
    ../../third_party/v8/src/detachable-vector.h
    ../../third_party/v8/src/diagnostics/code-tracer.h
    ../../third_party/v8/src/diagnostics/crash-key-noop.cc
    ../../third_party/v8/src/diagnostics/crash-key.h
    ../../third_party/v8/src/disasm.h
    ../../third_party/v8/src/disassembler.cc
    ../../third_party/v8/src/disassembler.h
    ../../third_party/v8/src/diy-fp.cc
    ../../third_party/v8/src/diy-fp.h
    ../../third_party/v8/src/double.h
    ../../third_party/v8/src/dtoa.cc
    ../../third_party/v8/src/dtoa.h
    ../../third_party/v8/src/eh-frame.cc
    ../../third_party/v8/src/eh-frame.h
    ../../third_party/v8/src/elements-inl.h
    ../../third_party/v8/src/elements-kind.cc
    ../../third_party/v8/src/elements-kind.h
    ../../third_party/v8/src/elements.cc
    ../../third_party/v8/src/elements.h
    ../../third_party/v8/src/execution.cc
    ../../third_party/v8/src/execution.h
    ../../third_party/v8/src/execution/execution.h
    ../../third_party/v8/src/execution/frame-constants.h
    ../../third_party/v8/src/execution/frames.h
    ../../third_party/v8/src/execution/futex-emulation.h
    ../../third_party/v8/src/execution/isolate-data.h
    ../../third_party/v8/src/execution/isolate-inl.h
    ../../third_party/v8/src/execution/isolate.h
    ../../third_party/v8/src/execution/message-template.h
    ../../third_party/v8/src/execution/messages.h
    ../../third_party/v8/src/execution/thread-id.h
    ../../third_party/v8/src/execution/thread-local-top.h
    ../../third_party/v8/src/execution/v8threads.h
    ../../third_party/v8/src/execution/x64/frame-constants-x64.h
    ../../third_party/v8/src/extensions/externalize-string-extension.cc
    ../../third_party/v8/src/extensions/externalize-string-extension.h
    ../../third_party/v8/src/extensions/free-buffer-extension.cc
    ../../third_party/v8/src/extensions/free-buffer-extension.h
    ../../third_party/v8/src/extensions/gc-extension.cc
    ../../third_party/v8/src/extensions/gc-extension.h
    ../../third_party/v8/src/extensions/ignition-statistics-extension.cc
    ../../third_party/v8/src/extensions/ignition-statistics-extension.h
    ../../third_party/v8/src/extensions/statistics-extension.cc
    ../../third_party/v8/src/extensions/statistics-extension.h
    ../../third_party/v8/src/extensions/trigger-failure-extension.cc
    ../../third_party/v8/src/extensions/trigger-failure-extension.h
    ../../third_party/v8/src/external-reference-table.cc
    ../../third_party/v8/src/external-reference-table.h
    ../../third_party/v8/src/external-reference.cc
    ../../third_party/v8/src/external-reference.h
    ../../third_party/v8/src/fast-dtoa.cc
    ../../third_party/v8/src/fast-dtoa.h
    ../../third_party/v8/src/feedback-vector-inl.h
    ../../third_party/v8/src/feedback-vector.cc
    ../../third_party/v8/src/feedback-vector.h
    ../../third_party/v8/src/field-index-inl.h
    ../../third_party/v8/src/field-index.h
    ../../third_party/v8/src/field-type.cc
    ../../third_party/v8/src/field-type.h
    ../../third_party/v8/src/fixed-dtoa.cc
    ../../third_party/v8/src/fixed-dtoa.h
    ../../third_party/v8/src/flag-definitions.h
    ../../third_party/v8/src/flags.cc
    ../../third_party/v8/src/flags.h
    ../../third_party/v8/src/flags/flag-definitions.h
    ../../third_party/v8/src/flags/flags.h
    ../../third_party/v8/src/flush-instruction-cache.cc
    ../../third_party/v8/src/flush-instruction-cache.h
    ../../third_party/v8/src/frame-constants.h
    ../../third_party/v8/src/frames-inl.h
    ../../third_party/v8/src/frames.cc
    ../../third_party/v8/src/frames.h
    ../../third_party/v8/src/function-kind.h
    ../../third_party/v8/src/futex-emulation.cc
    ../../third_party/v8/src/futex-emulation.h
    ../../third_party/v8/src/gdb-jit.cc
    ../../third_party/v8/src/gdb-jit.h
    ../../third_party/v8/src/global-handles.cc
    ../../third_party/v8/src/global-handles.h
    ../../third_party/v8/src/globals.h
    ../../third_party/v8/src/handler-table.cc
    ../../third_party/v8/src/handler-table.h
    ../../third_party/v8/src/handles-inl.h
    ../../third_party/v8/src/handles.cc
    ../../third_party/v8/src/handles.h
    ../../third_party/v8/src/handles/global-handles.h
    ../../third_party/v8/src/handles/handles-inl.h
    ../../third_party/v8/src/handles/handles.h
    ../../third_party/v8/src/handles/maybe-handles-inl.h
    ../../third_party/v8/src/handles/maybe-handles.h
    ../../third_party/v8/src/hash-seed-inl.h
    ../../third_party/v8/src/heap-symbols.h
    ../../third_party/v8/src/heap/array-buffer-collector.cc
    ../../third_party/v8/src/heap/array-buffer-collector.h
    ../../third_party/v8/src/heap/array-buffer-tracker-inl.h
    ../../third_party/v8/src/heap/array-buffer-tracker.cc
    ../../third_party/v8/src/heap/array-buffer-tracker.h
    ../../third_party/v8/src/heap/barrier.h
    ../../third_party/v8/src/heap/code-stats.cc
    ../../third_party/v8/src/heap/code-stats.h
    ../../third_party/v8/src/heap/concurrent-marking.cc
    ../../third_party/v8/src/heap/concurrent-marking.h
    ../../third_party/v8/src/heap/embedder-tracing.cc
    ../../third_party/v8/src/heap/embedder-tracing.h
    ../../third_party/v8/src/heap/factory-inl.h
    ../../third_party/v8/src/heap/factory.cc
    ../../third_party/v8/src/heap/factory.h
    ../../third_party/v8/src/heap/gc-idle-time-handler.cc
    ../../third_party/v8/src/heap/gc-idle-time-handler.h
    ../../third_party/v8/src/heap/gc-tracer.cc
    ../../third_party/v8/src/heap/gc-tracer.h
    ../../third_party/v8/src/heap/heap-controller.cc
    ../../third_party/v8/src/heap/heap-controller.h
    ../../third_party/v8/src/heap/heap-inl.h
    ../../third_party/v8/src/heap/heap-write-barrier-inl.h
    ../../third_party/v8/src/heap/heap-write-barrier.h
    ../../third_party/v8/src/heap/heap.cc
    ../../third_party/v8/src/heap/heap.h
    ../../third_party/v8/src/heap/incremental-marking-inl.h
    ../../third_party/v8/src/heap/incremental-marking-job.cc
    ../../third_party/v8/src/heap/incremental-marking-job.h
    ../../third_party/v8/src/heap/incremental-marking.cc
    ../../third_party/v8/src/heap/incremental-marking.h
    ../../third_party/v8/src/heap/invalidated-slots-inl.h
    ../../third_party/v8/src/heap/invalidated-slots.cc
    ../../third_party/v8/src/heap/invalidated-slots.h
    ../../third_party/v8/src/heap/item-parallel-job.cc
    ../../third_party/v8/src/heap/item-parallel-job.h
    ../../third_party/v8/src/heap/local-allocator-inl.h
    ../../third_party/v8/src/heap/local-allocator.h
    ../../third_party/v8/src/heap/mark-compact-inl.h
    ../../third_party/v8/src/heap/mark-compact.cc
    ../../third_party/v8/src/heap/mark-compact.h
    ../../third_party/v8/src/heap/marking.cc
    ../../third_party/v8/src/heap/marking.h
    ../../third_party/v8/src/heap/memory-reducer.cc
    ../../third_party/v8/src/heap/memory-reducer.h
    ../../third_party/v8/src/heap/object-stats.cc
    ../../third_party/v8/src/heap/object-stats.h
    ../../third_party/v8/src/heap/objects-visiting-inl.h
    ../../third_party/v8/src/heap/objects-visiting.cc
    ../../third_party/v8/src/heap/objects-visiting.h
    ../../third_party/v8/src/heap/read-only-heap-inl.h
    ../../third_party/v8/src/heap/read-only-heap.cc
    ../../third_party/v8/src/heap/read-only-heap.h
    ../../third_party/v8/src/heap/remembered-set.h
    ../../third_party/v8/src/heap/scavenge-job.cc
    ../../third_party/v8/src/heap/scavenge-job.h
    ../../third_party/v8/src/heap/scavenger-inl.h
    ../../third_party/v8/src/heap/scavenger.cc
    ../../third_party/v8/src/heap/scavenger.h
    ../../third_party/v8/src/heap/setup-heap-internal.cc
    ../../third_party/v8/src/heap/slot-set.cc
    ../../third_party/v8/src/heap/slot-set.h
    ../../third_party/v8/src/heap/spaces-inl.h
    ../../third_party/v8/src/heap/spaces.cc
    ../../third_party/v8/src/heap/spaces.h
    ../../third_party/v8/src/heap/store-buffer-inl.h
    ../../third_party/v8/src/heap/store-buffer.cc
    ../../third_party/v8/src/heap/store-buffer.h
    ../../third_party/v8/src/heap/stress-marking-observer.cc
    ../../third_party/v8/src/heap/stress-marking-observer.h
    ../../third_party/v8/src/heap/stress-scavenge-observer.cc
    ../../third_party/v8/src/heap/stress-scavenge-observer.h
    ../../third_party/v8/src/heap/sweeper.cc
    ../../third_party/v8/src/heap/sweeper.h
    ../../third_party/v8/src/heap/worklist.h
    ../../third_party/v8/src/ic/accessor-assembler.cc
    ../../third_party/v8/src/ic/accessor-assembler.h
    ../../third_party/v8/src/ic/binary-op-assembler.cc
    ../../third_party/v8/src/ic/binary-op-assembler.h
    ../../third_party/v8/src/ic/call-optimization.cc
    ../../third_party/v8/src/ic/call-optimization.h
    ../../third_party/v8/src/ic/handler-configuration-inl.h
    ../../third_party/v8/src/ic/handler-configuration.cc
    ../../third_party/v8/src/ic/handler-configuration.h
    ../../third_party/v8/src/ic/ic-inl.h
    ../../third_party/v8/src/ic/ic-stats.cc
    ../../third_party/v8/src/ic/ic-stats.h
    ../../third_party/v8/src/ic/ic.cc
    ../../third_party/v8/src/ic/ic.h
    ../../third_party/v8/src/ic/keyed-store-generic.cc
    ../../third_party/v8/src/ic/keyed-store-generic.h
    ../../third_party/v8/src/ic/stub-cache.cc
    ../../third_party/v8/src/ic/stub-cache.h
    ../../third_party/v8/src/icu_util.cc
    ../../third_party/v8/src/icu_util.h
    ../../third_party/v8/src/identity-map.cc
    ../../third_party/v8/src/identity-map.h
    ../../third_party/v8/src/init/heap-symbols.h
    ../../third_party/v8/src/init/isolate-allocator.h
    ../../third_party/v8/src/init/setup-isolate-full.cc
    ../../third_party/v8/src/init/setup-isolate.h
    ../../third_party/v8/src/init/v8.h
    ../../third_party/v8/src/inspector/custom-preview.cc
    ../../third_party/v8/src/inspector/custom-preview.h
    ../../third_party/v8/src/inspector/injected-script.cc
    ../../third_party/v8/src/inspector/injected-script.h
    ../../third_party/v8/src/inspector/inspected-context.cc
    ../../third_party/v8/src/inspector/inspected-context.h
    ../../third_party/v8/src/inspector/remote-object-id.cc
    ../../third_party/v8/src/inspector/remote-object-id.h
    ../../third_party/v8/src/inspector/search-util.cc
    ../../third_party/v8/src/inspector/search-util.h
    ../../third_party/v8/src/inspector/string-16.cc
    ../../third_party/v8/src/inspector/string-16.h
    ../../third_party/v8/src/inspector/string-util.cc
    ../../third_party/v8/src/inspector/string-util.h
    ../../third_party/v8/src/inspector/test-interface.cc
    ../../third_party/v8/src/inspector/test-interface.h
    ../../third_party/v8/src/inspector/v8-console-agent-impl.cc
    ../../third_party/v8/src/inspector/v8-console-agent-impl.h
    ../../third_party/v8/src/inspector/v8-console-message.cc
    ../../third_party/v8/src/inspector/v8-console-message.h
    ../../third_party/v8/src/inspector/v8-console.cc
    ../../third_party/v8/src/inspector/v8-console.h
    ../../third_party/v8/src/inspector/v8-debugger-agent-impl.cc
    ../../third_party/v8/src/inspector/v8-debugger-agent-impl.h
    ../../third_party/v8/src/inspector/v8-debugger-script.cc
    ../../third_party/v8/src/inspector/v8-debugger-script.h
    ../../third_party/v8/src/inspector/v8-debugger.cc
    ../../third_party/v8/src/inspector/v8-debugger.h
    ../../third_party/v8/src/inspector/v8-heap-profiler-agent-impl.cc
    ../../third_party/v8/src/inspector/v8-heap-profiler-agent-impl.h
    ../../third_party/v8/src/inspector/v8-inspector-impl.cc
    ../../third_party/v8/src/inspector/v8-inspector-impl.h
    ../../third_party/v8/src/inspector/v8-inspector-protocol-encoding.cc
    ../../third_party/v8/src/inspector/v8-inspector-protocol-encoding.h
    ../../third_party/v8/src/inspector/v8-inspector-session-impl.cc
    ../../third_party/v8/src/inspector/v8-inspector-session-impl.h
    ../../third_party/v8/src/inspector/v8-profiler-agent-impl.cc
    ../../third_party/v8/src/inspector/v8-profiler-agent-impl.h
    ../../third_party/v8/src/inspector/v8-regex.cc
    ../../third_party/v8/src/inspector/v8-regex.h
    ../../third_party/v8/src/inspector/v8-runtime-agent-impl.cc
    ../../third_party/v8/src/inspector/v8-runtime-agent-impl.h
    ../../third_party/v8/src/inspector/v8-schema-agent-impl.cc
    ../../third_party/v8/src/inspector/v8-schema-agent-impl.h
    ../../third_party/v8/src/inspector/v8-stack-trace-impl.cc
    ../../third_party/v8/src/inspector/v8-stack-trace-impl.h
    ../../third_party/v8/src/inspector/v8-string-conversions.cc
    ../../third_party/v8/src/inspector/v8-string-conversions.h
    ../../third_party/v8/src/inspector/v8-value-utils.cc
    ../../third_party/v8/src/inspector/v8-value-utils.h
    ../../third_party/v8/src/inspector/value-mirror.cc
    ../../third_party/v8/src/inspector/value-mirror.h
    ../../third_party/v8/src/inspector/wasm-translation.cc
    ../../third_party/v8/src/inspector/wasm-translation.h
    ../../third_party/v8/src/interface-descriptors.cc
    ../../third_party/v8/src/interface-descriptors.h
    ../../third_party/v8/src/interpreter/block-coverage-builder.h
    ../../third_party/v8/src/interpreter/bytecode-array-accessor.cc
    ../../third_party/v8/src/interpreter/bytecode-array-accessor.h
    ../../third_party/v8/src/interpreter/bytecode-array-builder.cc
    ../../third_party/v8/src/interpreter/bytecode-array-builder.h
    ../../third_party/v8/src/interpreter/bytecode-array-iterator.cc
    ../../third_party/v8/src/interpreter/bytecode-array-iterator.h
    ../../third_party/v8/src/interpreter/bytecode-array-random-iterator.cc
    ../../third_party/v8/src/interpreter/bytecode-array-random-iterator.h
    ../../third_party/v8/src/interpreter/bytecode-array-writer.cc
    ../../third_party/v8/src/interpreter/bytecode-array-writer.h
    ../../third_party/v8/src/interpreter/bytecode-decoder.cc
    ../../third_party/v8/src/interpreter/bytecode-decoder.h
    ../../third_party/v8/src/interpreter/bytecode-flags.cc
    ../../third_party/v8/src/interpreter/bytecode-flags.h
    ../../third_party/v8/src/interpreter/bytecode-generator.cc
    ../../third_party/v8/src/interpreter/bytecode-generator.h
    ../../third_party/v8/src/interpreter/bytecode-jump-table.h
    ../../third_party/v8/src/interpreter/bytecode-label.cc
    ../../third_party/v8/src/interpreter/bytecode-label.h
    ../../third_party/v8/src/interpreter/bytecode-node.cc
    ../../third_party/v8/src/interpreter/bytecode-node.h
    ../../third_party/v8/src/interpreter/bytecode-operands.cc
    ../../third_party/v8/src/interpreter/bytecode-operands.h
    ../../third_party/v8/src/interpreter/bytecode-register-allocator.h
    ../../third_party/v8/src/interpreter/bytecode-register-optimizer.cc
    ../../third_party/v8/src/interpreter/bytecode-register-optimizer.h
    ../../third_party/v8/src/interpreter/bytecode-register.cc
    ../../third_party/v8/src/interpreter/bytecode-register.h
    ../../third_party/v8/src/interpreter/bytecode-source-info.cc
    ../../third_party/v8/src/interpreter/bytecode-source-info.h
    ../../third_party/v8/src/interpreter/bytecode-traits.h
    ../../third_party/v8/src/interpreter/bytecodes.cc
    ../../third_party/v8/src/interpreter/bytecodes.h
    ../../third_party/v8/src/interpreter/constant-array-builder.cc
    ../../third_party/v8/src/interpreter/constant-array-builder.h
    ../../third_party/v8/src/interpreter/control-flow-builders.cc
    ../../third_party/v8/src/interpreter/control-flow-builders.h
    ../../third_party/v8/src/interpreter/handler-table-builder.cc
    ../../third_party/v8/src/interpreter/handler-table-builder.h
    ../../third_party/v8/src/interpreter/interpreter-assembler.cc
    ../../third_party/v8/src/interpreter/interpreter-assembler.h
    ../../third_party/v8/src/interpreter/interpreter-generator.cc
    ../../third_party/v8/src/interpreter/interpreter-generator.h
    ../../third_party/v8/src/interpreter/interpreter-intrinsics-generator.cc
    ../../third_party/v8/src/interpreter/interpreter-intrinsics-generator.h
    ../../third_party/v8/src/interpreter/interpreter-intrinsics.cc
    ../../third_party/v8/src/interpreter/interpreter-intrinsics.h
    ../../third_party/v8/src/interpreter/interpreter.cc
    ../../third_party/v8/src/interpreter/interpreter.h
    ../../third_party/v8/src/isolate-allocator.cc
    ../../third_party/v8/src/isolate-allocator.h
    ../../third_party/v8/src/isolate-data.h
    ../../third_party/v8/src/isolate-inl.h
    ../../third_party/v8/src/isolate.cc
    ../../third_party/v8/src/isolate.h
    ../../third_party/v8/src/json-parser.cc
    ../../third_party/v8/src/json-parser.h
    ../../third_party/v8/src/json-stringifier.cc
    ../../third_party/v8/src/json-stringifier.h
    ../../third_party/v8/src/keys.cc
    ../../third_party/v8/src/keys.h
    ../../third_party/v8/src/label.h
    ../../third_party/v8/src/layout-descriptor-inl.h
    ../../third_party/v8/src/layout-descriptor.cc
    ../../third_party/v8/src/layout-descriptor.h
    ../../third_party/v8/src/libplatform/default-foreground-task-runner.cc
    ../../third_party/v8/src/libplatform/default-foreground-task-runner.h
    ../../third_party/v8/src/libplatform/default-platform.cc
    ../../third_party/v8/src/libplatform/default-platform.h
    ../../third_party/v8/src/libplatform/default-worker-threads-task-runner.cc
    ../../third_party/v8/src/libplatform/default-worker-threads-task-runner.h
    ../../third_party/v8/src/libplatform/delayed-task-queue.cc
    ../../third_party/v8/src/libplatform/delayed-task-queue.h
    ../../third_party/v8/src/libplatform/task-queue.cc
    ../../third_party/v8/src/libplatform/task-queue.h
    ../../third_party/v8/src/libplatform/tracing/trace-buffer.cc
    ../../third_party/v8/src/libplatform/tracing/trace-buffer.h
    ../../third_party/v8/src/libplatform/tracing/trace-config.cc
    ../../third_party/v8/src/libplatform/tracing/trace-object.cc
    ../../third_party/v8/src/libplatform/tracing/trace-writer.cc
    ../../third_party/v8/src/libplatform/tracing/trace-writer.h
    ../../third_party/v8/src/libplatform/tracing/tracing-controller.cc
    ../../third_party/v8/src/libplatform/worker-thread.cc
    ../../third_party/v8/src/libplatform/worker-thread.h
    ../../third_party/v8/src/libsampler/sampler.cc
    ../../third_party/v8/src/libsampler/sampler.h
    ../../third_party/v8/src/locked-queue-inl.h
    ../../third_party/v8/src/locked-queue.h
    ../../third_party/v8/src/log-inl.h
    ../../third_party/v8/src/log-utils.cc
    ../../third_party/v8/src/log-utils.h
    ../../third_party/v8/src/log.cc
    ../../third_party/v8/src/log.h
    ../../third_party/v8/src/logging/code-events.h
    ../../third_party/v8/src/logging/counters-definitions.h
    ../../third_party/v8/src/logging/log.h
    ../../third_party/v8/src/lookup-cache-inl.h
    ../../third_party/v8/src/lookup-cache.cc
    ../../third_party/v8/src/lookup-cache.h
    ../../third_party/v8/src/lookup-inl.h
    ../../third_party/v8/src/lookup.cc
    ../../third_party/v8/src/lookup.h
    ../../third_party/v8/src/machine-type.cc
    ../../third_party/v8/src/machine-type.h
    ../../third_party/v8/src/macro-assembler-inl.h
    ../../third_party/v8/src/macro-assembler.h
    ../../third_party/v8/src/map-updater.cc
    ../../third_party/v8/src/map-updater.h
    ../../third_party/v8/src/math-random.cc
    ../../third_party/v8/src/math-random.h
    ../../third_party/v8/src/maybe-handles-inl.h
    ../../third_party/v8/src/maybe-handles.h
    ../../third_party/v8/src/memcopy.cc
    ../../third_party/v8/src/memcopy.h
    ../../third_party/v8/src/message-template.h
    ../../third_party/v8/src/messages.cc
    ../../third_party/v8/src/messages.h
    ../../third_party/v8/src/microtask-queue.cc
    ../../third_party/v8/src/microtask-queue.h
    ../../third_party/v8/src/msan.h
    ../../third_party/v8/src/numbers/conversions-inl.h
    ../../third_party/v8/src/numbers/conversions.h
    ../../third_party/v8/src/numbers/diy-fp.h
    ../../third_party/v8/src/numbers/double.h
    ../../third_party/v8/src/numbers/hash-seed-inl.h
    ../../third_party/v8/src/objects-body-descriptors-inl.h
    ../../third_party/v8/src/objects-body-descriptors.h
    ../../third_party/v8/src/objects-debug.cc
    ../../third_party/v8/src/objects-definitions.h
    ../../third_party/v8/src/objects-inl.h
    ../../third_party/v8/src/objects-printer.cc
    ../../third_party/v8/src/objects.cc
    ../../third_party/v8/src/objects.h
    ../../third_party/v8/src/objects/allocation-site-inl.h
    ../../third_party/v8/src/objects/allocation-site.h
    ../../third_party/v8/src/objects/api-callbacks-inl.h
    ../../third_party/v8/src/objects/api-callbacks.h
    ../../third_party/v8/src/objects/arguments-inl.h
    ../../third_party/v8/src/objects/arguments.h
    ../../third_party/v8/src/objects/bigint.cc
    ../../third_party/v8/src/objects/bigint.h
    ../../third_party/v8/src/objects/cell-inl.h
    ../../third_party/v8/src/objects/cell.h
    ../../third_party/v8/src/objects/code-inl.h
    ../../third_party/v8/src/objects/code.cc
    ../../third_party/v8/src/objects/code.h
    ../../third_party/v8/src/objects/compilation-cache-inl.h
    ../../third_party/v8/src/objects/compilation-cache.h
    ../../third_party/v8/src/objects/compressed-slots-inl.h
    ../../third_party/v8/src/objects/compressed-slots.h
    ../../third_party/v8/src/objects/contexts-inl.h
    ../../third_party/v8/src/objects/contexts.h
    ../../third_party/v8/src/objects/data-handler-inl.h
    ../../third_party/v8/src/objects/data-handler.h
    ../../third_party/v8/src/objects/debug-objects-inl.h
    ../../third_party/v8/src/objects/debug-objects.cc
    ../../third_party/v8/src/objects/debug-objects.h
    ../../third_party/v8/src/objects/descriptor-array-inl.h
    ../../third_party/v8/src/objects/descriptor-array.h
    ../../third_party/v8/src/objects/dictionary-inl.h
    ../../third_party/v8/src/objects/dictionary.h
    ../../third_party/v8/src/objects/elements-kind.h
    ../../third_party/v8/src/objects/embedder-data-array-inl.h
    ../../third_party/v8/src/objects/embedder-data-array.cc
    ../../third_party/v8/src/objects/embedder-data-array.h
    ../../third_party/v8/src/objects/embedder-data-slot-inl.h
    ../../third_party/v8/src/objects/embedder-data-slot.h
    ../../third_party/v8/src/objects/feedback-cell-inl.h
    ../../third_party/v8/src/objects/feedback-cell.h
    ../../third_party/v8/src/objects/feedback-vector-inl.h
    ../../third_party/v8/src/objects/feedback-vector.h
    ../../third_party/v8/src/objects/field-index-inl.h
    ../../third_party/v8/src/objects/field-index.h
    ../../third_party/v8/src/objects/field-type.h
    ../../third_party/v8/src/objects/fixed-array-inl.h
    ../../third_party/v8/src/objects/fixed-array.h
    ../../third_party/v8/src/objects/foreign-inl.h
    ../../third_party/v8/src/objects/foreign.h
    ../../third_party/v8/src/objects/frame-array-inl.h
    ../../third_party/v8/src/objects/frame-array.h
    ../../third_party/v8/src/objects/free-space-inl.h
    ../../third_party/v8/src/objects/free-space.h
    ../../third_party/v8/src/objects/function-kind.h
    ../../third_party/v8/src/objects/hash-table-inl.h
    ../../third_party/v8/src/objects/hash-table.h
    ../../third_party/v8/src/objects/heap-number-inl.h
    ../../third_party/v8/src/objects/heap-number.h
    ../../third_party/v8/src/objects/heap-object-inl.h
    ../../third_party/v8/src/objects/heap-object.h
    ../../third_party/v8/src/objects/instance-type-inl.h
    ../../third_party/v8/src/objects/instance-type.h
    ../../third_party/v8/src/objects/js-array-buffer-inl.h
    ../../third_party/v8/src/objects/js-array-buffer.cc
    ../../third_party/v8/src/objects/js-array-buffer.h
    ../../third_party/v8/src/objects/js-array-inl.h
    ../../third_party/v8/src/objects/js-array.h
    ../../third_party/v8/src/objects/js-collection-inl.h
    ../../third_party/v8/src/objects/js-collection-iterator.h
    ../../third_party/v8/src/objects/js-collection.h
    ../../third_party/v8/src/objects/js-generator-inl.h
    ../../third_party/v8/src/objects/js-generator.h
    ../../third_party/v8/src/objects/js-objects-inl.h
    ../../third_party/v8/src/objects/js-objects.cc
    ../../third_party/v8/src/objects/js-objects.h
    ../../third_party/v8/src/objects/js-promise-inl.h
    ../../third_party/v8/src/objects/js-promise.h
    ../../third_party/v8/src/objects/js-proxy-inl.h
    ../../third_party/v8/src/objects/js-proxy.h
    ../../third_party/v8/src/objects/js-regexp-inl.h
    ../../third_party/v8/src/objects/js-regexp-string-iterator-inl.h
    ../../third_party/v8/src/objects/js-regexp-string-iterator.h
    ../../third_party/v8/src/objects/js-regexp.h
    ../../third_party/v8/src/objects/js-weak-refs-inl.h
    ../../third_party/v8/src/objects/js-weak-refs.h
    ../../third_party/v8/src/objects/keys.h
    ../../third_party/v8/src/objects/layout-descriptor-inl.h
    ../../third_party/v8/src/objects/layout-descriptor.h
    ../../third_party/v8/src/objects/literal-objects-inl.h
    ../../third_party/v8/src/objects/literal-objects.cc
    ../../third_party/v8/src/objects/literal-objects.h
    ../../third_party/v8/src/objects/lookup-cache-inl.h
    ../../third_party/v8/src/objects/lookup-cache.h
    ../../third_party/v8/src/objects/lookup-inl.h
    ../../third_party/v8/src/objects/lookup.h
    ../../third_party/v8/src/objects/managed.cc
    ../../third_party/v8/src/objects/managed.h
    ../../third_party/v8/src/objects/map-inl.h
    ../../third_party/v8/src/objects/map.cc
    ../../third_party/v8/src/objects/map.h
    ../../third_party/v8/src/objects/maybe-object-inl.h
    ../../third_party/v8/src/objects/maybe-object.h
    ../../third_party/v8/src/objects/microtask-inl.h
    ../../third_party/v8/src/objects/microtask.h
    ../../third_party/v8/src/objects/module-inl.h
    ../../third_party/v8/src/objects/module.cc
    ../../third_party/v8/src/objects/module.h
    ../../third_party/v8/src/objects/name-inl.h
    ../../third_party/v8/src/objects/name.h
    ../../third_party/v8/src/objects/object-list-macros.h
    ../../third_party/v8/src/objects/object-macros-undef.h
    ../../third_party/v8/src/objects/object-macros.h
    ../../third_party/v8/src/objects/objects-definitions.h
    ../../third_party/v8/src/objects/objects-inl.h
    ../../third_party/v8/src/objects/objects.h
    ../../third_party/v8/src/objects/oddball-inl.h
    ../../third_party/v8/src/objects/oddball.h
    ../../third_party/v8/src/objects/ordered-hash-table-inl.h
    ../../third_party/v8/src/objects/ordered-hash-table.cc
    ../../third_party/v8/src/objects/ordered-hash-table.h
    ../../third_party/v8/src/objects/promise-inl.h
    ../../third_party/v8/src/objects/promise.h
    ../../third_party/v8/src/objects/property-array-inl.h
    ../../third_party/v8/src/objects/property-array.h
    ../../third_party/v8/src/objects/property-cell-inl.h
    ../../third_party/v8/src/objects/property-cell.h
    ../../third_party/v8/src/objects/property-descriptor-object-inl.h
    ../../third_party/v8/src/objects/property-descriptor-object.h
    ../../third_party/v8/src/objects/property-details.h
    ../../third_party/v8/src/objects/property.h
    ../../third_party/v8/src/objects/prototype-info-inl.h
    ../../third_party/v8/src/objects/prototype-info.h
    ../../third_party/v8/src/objects/prototype-inl.h
    ../../third_party/v8/src/objects/prototype.h
    ../../third_party/v8/src/objects/regexp-match-info.h
    ../../third_party/v8/src/objects/scope-info.cc
    ../../third_party/v8/src/objects/scope-info.h
    ../../third_party/v8/src/objects/script-inl.h
    ../../third_party/v8/src/objects/script.h
    ../../third_party/v8/src/objects/shared-function-info-inl.h
    ../../third_party/v8/src/objects/shared-function-info.h
    ../../third_party/v8/src/objects/slots-atomic-inl.h
    ../../third_party/v8/src/objects/slots-inl.h
    ../../third_party/v8/src/objects/slots.h
    ../../third_party/v8/src/objects/smi-inl.h
    ../../third_party/v8/src/objects/smi.h
    ../../third_party/v8/src/objects/stack-frame-info-inl.h
    ../../third_party/v8/src/objects/stack-frame-info.cc
    ../../third_party/v8/src/objects/stack-frame-info.h
    ../../third_party/v8/src/objects/string-comparator.cc
    ../../third_party/v8/src/objects/string-comparator.h
    ../../third_party/v8/src/objects/string-inl.h
    ../../third_party/v8/src/objects/string-table-inl.h
    ../../third_party/v8/src/objects/string-table.h
    ../../third_party/v8/src/objects/string.cc
    ../../third_party/v8/src/objects/string.h
    ../../third_party/v8/src/objects/struct-inl.h
    ../../third_party/v8/src/objects/struct.h
    ../../third_party/v8/src/objects/tagged-impl-inl.h
    ../../third_party/v8/src/objects/tagged-impl.h
    ../../third_party/v8/src/objects/template-objects-inl.h
    ../../third_party/v8/src/objects/template-objects.cc
    ../../third_party/v8/src/objects/template-objects.h
    ../../third_party/v8/src/objects/templates-inl.h
    ../../third_party/v8/src/objects/templates.h
    ../../third_party/v8/src/objects/transitions-inl.h
    ../../third_party/v8/src/objects/transitions.h
    ../../third_party/v8/src/objects/type-hints.h
    ../../third_party/v8/src/objects/visitors.h
    ../../third_party/v8/src/optimized-compilation-info.cc
    ../../third_party/v8/src/optimized-compilation-info.h
    ../../third_party/v8/src/ostreams.cc
    ../../third_party/v8/src/ostreams.h
    ../../third_party/v8/src/parsing/expression-scope-reparenter.cc
    ../../third_party/v8/src/parsing/expression-scope-reparenter.h
    ../../third_party/v8/src/parsing/expression-scope.h
    ../../third_party/v8/src/parsing/func-name-inferrer.cc
    ../../third_party/v8/src/parsing/func-name-inferrer.h
    ../../third_party/v8/src/parsing/keywords-gen.h
    ../../third_party/v8/src/parsing/parse-info.cc
    ../../third_party/v8/src/parsing/parse-info.h
    ../../third_party/v8/src/parsing/parser-base.h
    ../../third_party/v8/src/parsing/parser.cc
    ../../third_party/v8/src/parsing/parser.h
    ../../third_party/v8/src/parsing/parsing.cc
    ../../third_party/v8/src/parsing/parsing.h
    ../../third_party/v8/src/parsing/preparse-data-impl.h
    ../../third_party/v8/src/parsing/preparse-data.cc
    ../../third_party/v8/src/parsing/preparse-data.h
    ../../third_party/v8/src/parsing/preparser-logger.h
    ../../third_party/v8/src/parsing/preparser.cc
    ../../third_party/v8/src/parsing/preparser.h
    ../../third_party/v8/src/parsing/rewriter.cc
    ../../third_party/v8/src/parsing/rewriter.h
    ../../third_party/v8/src/parsing/scanner-character-streams.cc
    ../../third_party/v8/src/parsing/scanner-character-streams.h
    ../../third_party/v8/src/parsing/scanner-inl.h
    ../../third_party/v8/src/parsing/scanner.cc
    ../../third_party/v8/src/parsing/scanner.h
    ../../third_party/v8/src/parsing/token.cc
    ../../third_party/v8/src/parsing/token.h
    ../../third_party/v8/src/pending-compilation-error-handler.cc
    ../../third_party/v8/src/pending-compilation-error-handler.h
    ../../third_party/v8/src/perf-jit.cc
    ../../third_party/v8/src/perf-jit.h
    ../../third_party/v8/src/pointer-with-payload.h
    ../../third_party/v8/src/profiler/allocation-tracker.cc
    ../../third_party/v8/src/profiler/allocation-tracker.h
    ../../third_party/v8/src/profiler/circular-queue-inl.h
    ../../third_party/v8/src/profiler/circular-queue.h
    ../../third_party/v8/src/profiler/cpu-profiler-inl.h
    ../../third_party/v8/src/profiler/cpu-profiler.cc
    ../../third_party/v8/src/profiler/cpu-profiler.h
    ../../third_party/v8/src/profiler/heap-profiler.cc
    ../../third_party/v8/src/profiler/heap-profiler.h
    ../../third_party/v8/src/profiler/heap-snapshot-generator-inl.h
    ../../third_party/v8/src/profiler/heap-snapshot-generator.cc
    ../../third_party/v8/src/profiler/heap-snapshot-generator.h
    ../../third_party/v8/src/profiler/profile-generator-inl.h
    ../../third_party/v8/src/profiler/profile-generator.cc
    ../../third_party/v8/src/profiler/profile-generator.h
    ../../third_party/v8/src/profiler/profiler-listener.cc
    ../../third_party/v8/src/profiler/profiler-listener.h
    ../../third_party/v8/src/profiler/sampling-heap-profiler.cc
    ../../third_party/v8/src/profiler/sampling-heap-profiler.h
    ../../third_party/v8/src/profiler/strings-storage.cc
    ../../third_party/v8/src/profiler/strings-storage.h
    ../../third_party/v8/src/profiler/tick-sample.cc
    ../../third_party/v8/src/profiler/tick-sample.h
    ../../third_party/v8/src/profiler/tracing-cpu-profiler.cc
    ../../third_party/v8/src/profiler/tracing-cpu-profiler.h
    ../../third_party/v8/src/property-descriptor.cc
    ../../third_party/v8/src/property-descriptor.h
    ../../third_party/v8/src/property-details.h
    ../../third_party/v8/src/property.cc
    ../../third_party/v8/src/property.h
    ../../third_party/v8/src/prototype-inl.h
    ../../third_party/v8/src/prototype.h
    ../../third_party/v8/src/ptr-compr-inl.h
    ../../third_party/v8/src/ptr-compr.h
    ../../third_party/v8/src/regexp/bytecodes-irregexp.h
    ../../third_party/v8/src/regexp/interpreter-irregexp.cc
    ../../third_party/v8/src/regexp/interpreter-irregexp.h
    ../../third_party/v8/src/regexp/jsregexp-inl.h
    ../../third_party/v8/src/regexp/jsregexp.cc
    ../../third_party/v8/src/regexp/jsregexp.h
    ../../third_party/v8/src/regexp/property-sequences.cc
    ../../third_party/v8/src/regexp/property-sequences.h
    ../../third_party/v8/src/regexp/regexp-ast.cc
    ../../third_party/v8/src/regexp/regexp-ast.h
    ../../third_party/v8/src/regexp/regexp-macro-assembler-irregexp-inl.h
    ../../third_party/v8/src/regexp/regexp-macro-assembler-irregexp.cc
    ../../third_party/v8/src/regexp/regexp-macro-assembler-irregexp.h
    ../../third_party/v8/src/regexp/regexp-macro-assembler-tracer.cc
    ../../third_party/v8/src/regexp/regexp-macro-assembler-tracer.h
    ../../third_party/v8/src/regexp/regexp-macro-assembler.cc
    ../../third_party/v8/src/regexp/regexp-macro-assembler.h
    ../../third_party/v8/src/regexp/regexp-parser.cc
    ../../third_party/v8/src/regexp/regexp-parser.h
    ../../third_party/v8/src/regexp/regexp-stack.cc
    ../../third_party/v8/src/regexp/regexp-stack.h
    ../../third_party/v8/src/regexp/regexp-utils.cc
    ../../third_party/v8/src/regexp/regexp-utils.h
    ../../third_party/v8/src/regexp/x64/regexp-macro-assembler-x64.cc
    ../../third_party/v8/src/regexp/x64/regexp-macro-assembler-x64.h
    ../../third_party/v8/src/register-arch.h
    ../../third_party/v8/src/register-configuration.cc
    ../../third_party/v8/src/register-configuration.h
    ../../third_party/v8/src/register.h
    ../../third_party/v8/src/reglist.h
    ../../third_party/v8/src/reloc-info.cc
    ../../third_party/v8/src/reloc-info.h
    ../../third_party/v8/src/roots-inl.h
    ../../third_party/v8/src/roots.cc
    ../../third_party/v8/src/roots.h
    ../../third_party/v8/src/roots/roots-inl.h
    ../../third_party/v8/src/roots/roots.h
    ../../third_party/v8/src/runtime-profiler.cc
    ../../third_party/v8/src/runtime-profiler.h
    ../../third_party/v8/src/runtime/runtime-array.cc
    ../../third_party/v8/src/runtime/runtime-atomics.cc
    ../../third_party/v8/src/runtime/runtime-bigint.cc
    ../../third_party/v8/src/runtime/runtime-classes.cc
    ../../third_party/v8/src/runtime/runtime-collections.cc
    ../../third_party/v8/src/runtime/runtime-compiler.cc
    ../../third_party/v8/src/runtime/runtime-date.cc
    ../../third_party/v8/src/runtime/runtime-debug.cc
    ../../third_party/v8/src/runtime/runtime-forin.cc
    ../../third_party/v8/src/runtime/runtime-function.cc
    ../../third_party/v8/src/runtime/runtime-futex.cc
    ../../third_party/v8/src/runtime/runtime-generator.cc
    ../../third_party/v8/src/runtime/runtime-internal.cc
    ../../third_party/v8/src/runtime/runtime-interpreter.cc
    ../../third_party/v8/src/runtime/runtime-literals.cc
    ../../third_party/v8/src/runtime/runtime-module.cc
    ../../third_party/v8/src/runtime/runtime-numbers.cc
    ../../third_party/v8/src/runtime/runtime-object.cc
    ../../third_party/v8/src/runtime/runtime-operators.cc
    ../../third_party/v8/src/runtime/runtime-promise.cc
    ../../third_party/v8/src/runtime/runtime-proxy.cc
    ../../third_party/v8/src/runtime/runtime-regexp.cc
    ../../third_party/v8/src/runtime/runtime-scopes.cc
    ../../third_party/v8/src/runtime/runtime-strings.cc
    ../../third_party/v8/src/runtime/runtime-symbol.cc
    ../../third_party/v8/src/runtime/runtime-test.cc
    ../../third_party/v8/src/runtime/runtime-typedarray.cc
    ../../third_party/v8/src/runtime/runtime-utils.h
    ../../third_party/v8/src/runtime/runtime-wasm.cc
    ../../third_party/v8/src/runtime/runtime-weak-refs.cc
    ../../third_party/v8/src/runtime/runtime.cc
    ../../third_party/v8/src/runtime/runtime.h
    ../../third_party/v8/src/safepoint-table.cc
    ../../third_party/v8/src/safepoint-table.h
    ../../third_party/v8/src/sanitizer/msan.h
    ../../third_party/v8/src/sanitizer/tsan.h
    ../../third_party/v8/src/setup-isolate-deserialize.cc
    ../../third_party/v8/src/setup-isolate.h
    ../../third_party/v8/src/signature.h
    ../../third_party/v8/src/simulator-base.cc
    ../../third_party/v8/src/simulator-base.h
    ../../third_party/v8/src/simulator.h
    ../../third_party/v8/src/snapshot/code-serializer.cc
    ../../third_party/v8/src/snapshot/code-serializer.h
    ../../third_party/v8/src/snapshot/deserializer-allocator.cc
    ../../third_party/v8/src/snapshot/deserializer-allocator.h
    ../../third_party/v8/src/snapshot/deserializer.cc
    ../../third_party/v8/src/snapshot/deserializer.h
    ../../third_party/v8/src/snapshot/embedded-data.cc
    ../../third_party/v8/src/snapshot/embedded-data.h
    ../../third_party/v8/src/snapshot/embedded-empty.cc
    ../../third_party/v8/src/snapshot/embedded-file-writer.h
    ../../third_party/v8/src/snapshot/embedded/embedded-data.h
    ../../third_party/v8/src/snapshot/embedded/embedded-file-writer.cc
    ../../third_party/v8/src/snapshot/embedded/embedded-file-writer.h
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-aix.cc
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-aix.h
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-base.cc
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-base.h
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-generic.cc
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-generic.h
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-mac.cc
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-mac.h
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-win.cc
    ../../third_party/v8/src/snapshot/embedded/platform-embedded-file-writer-win.h
    ../../third_party/v8/src/snapshot/mksnapshot.cc
    ../../third_party/v8/src/snapshot/natives-common.cc
    ../../third_party/v8/src/snapshot/natives.h
    ../../third_party/v8/src/snapshot/object-deserializer.cc
    ../../third_party/v8/src/snapshot/object-deserializer.h
    ../../third_party/v8/src/snapshot/partial-deserializer.cc
    ../../third_party/v8/src/snapshot/partial-deserializer.h
    ../../third_party/v8/src/snapshot/partial-serializer.cc
    ../../third_party/v8/src/snapshot/partial-serializer.h
    ../../third_party/v8/src/snapshot/read-only-deserializer.cc
    ../../third_party/v8/src/snapshot/read-only-deserializer.h
    ../../third_party/v8/src/snapshot/read-only-serializer.cc
    ../../third_party/v8/src/snapshot/read-only-serializer.h
    ../../third_party/v8/src/snapshot/references.h
    ../../third_party/v8/src/snapshot/roots-serializer.cc
    ../../third_party/v8/src/snapshot/roots-serializer.h
    ../../third_party/v8/src/snapshot/serializer-allocator.cc
    ../../third_party/v8/src/snapshot/serializer-allocator.h
    ../../third_party/v8/src/snapshot/serializer-common.cc
    ../../third_party/v8/src/snapshot/serializer-common.h
    ../../third_party/v8/src/snapshot/serializer.cc
    ../../third_party/v8/src/snapshot/serializer.h
    ../../third_party/v8/src/snapshot/snapshot-common.cc
    ../../third_party/v8/src/snapshot/snapshot-empty.cc
    ../../third_party/v8/src/snapshot/snapshot-source-sink.cc
    ../../third_party/v8/src/snapshot/snapshot-source-sink.h
    ../../third_party/v8/src/snapshot/snapshot.h
    ../../third_party/v8/src/snapshot/startup-deserializer.cc
    ../../third_party/v8/src/snapshot/startup-deserializer.h
    ../../third_party/v8/src/snapshot/startup-serializer.cc
    ../../third_party/v8/src/snapshot/startup-serializer.h
    ../../third_party/v8/src/source-position-table.cc
    ../../third_party/v8/src/source-position-table.h
    ../../third_party/v8/src/source-position.cc
    ../../third_party/v8/src/source-position.h
    ../../third_party/v8/src/splay-tree-inl.h
    ../../third_party/v8/src/splay-tree.h
    ../../third_party/v8/src/startup-data-util.cc
    ../../third_party/v8/src/startup-data-util.h
    ../../third_party/v8/src/string-builder-inl.h
    ../../third_party/v8/src/string-builder.cc
    ../../third_party/v8/src/string-case.cc
    ../../third_party/v8/src/string-case.h
    ../../third_party/v8/src/string-constants.cc
    ../../third_party/v8/src/string-constants.h
    ../../third_party/v8/src/string-hasher-inl.h
    ../../third_party/v8/src/string-hasher.h
    ../../third_party/v8/src/string-search.h
    ../../third_party/v8/src/string-stream.cc
    ../../third_party/v8/src/string-stream.h
    ../../third_party/v8/src/strings/char-predicates-inl.h
    ../../third_party/v8/src/strings/char-predicates.h
    ../../third_party/v8/src/strings/string-hasher-inl.h
    ../../third_party/v8/src/strings/string-hasher.h
    ../../third_party/v8/src/strings/unicode-decoder.h
    ../../third_party/v8/src/strings/unicode.h
    ../../third_party/v8/src/strtod.cc
    ../../third_party/v8/src/strtod.h
    ../../third_party/v8/src/task-utils.cc
    ../../third_party/v8/src/task-utils.h
    ../../third_party/v8/src/tasks/cancelable-task.h
    ../../third_party/v8/src/third_party/siphash/halfsiphash.cc
    ../../third_party/v8/src/third_party/siphash/halfsiphash.h
    ../../third_party/v8/src/third_party/utf8-decoder/utf8-decoder.h
    ../../third_party/v8/src/third_party/valgrind/valgrind.h
    ../../third_party/v8/src/thread-id.cc
    ../../third_party/v8/src/thread-id.h
    ../../third_party/v8/src/thread-local-top.cc
    ../../third_party/v8/src/thread-local-top.h
    ../../third_party/v8/src/torque/ast.h
    ../../third_party/v8/src/torque/cfg.cc
    ../../third_party/v8/src/torque/cfg.h
    ../../third_party/v8/src/torque/constants.h
    ../../third_party/v8/src/torque/contextual.h
    ../../third_party/v8/src/torque/csa-generator.cc
    ../../third_party/v8/src/torque/csa-generator.h
    ../../third_party/v8/src/torque/declarable.cc
    ../../third_party/v8/src/torque/declarable.h
    ../../third_party/v8/src/torque/declaration-visitor.cc
    ../../third_party/v8/src/torque/declaration-visitor.h
    ../../third_party/v8/src/torque/declarations.cc
    ../../third_party/v8/src/torque/declarations.h
    ../../third_party/v8/src/torque/earley-parser.cc
    ../../third_party/v8/src/torque/earley-parser.h
    ../../third_party/v8/src/torque/global-context.h
    ../../third_party/v8/src/torque/implementation-visitor.cc
    ../../third_party/v8/src/torque/implementation-visitor.h
    ../../third_party/v8/src/torque/instructions.cc
    ../../third_party/v8/src/torque/instructions.h
    ../../third_party/v8/src/torque/parameter-difference.h
    ../../third_party/v8/src/torque/server-data.cc
    ../../third_party/v8/src/torque/server-data.h
    ../../third_party/v8/src/torque/source-positions.cc
    ../../third_party/v8/src/torque/source-positions.h
    ../../third_party/v8/src/torque/torque-compiler.cc
    ../../third_party/v8/src/torque/torque-compiler.h
    ../../third_party/v8/src/torque/torque-parser.cc
    ../../third_party/v8/src/torque/torque-parser.h
    ../../third_party/v8/src/torque/torque.cc
    ../../third_party/v8/src/torque/type-oracle.cc
    ../../third_party/v8/src/torque/type-oracle.h
    ../../third_party/v8/src/torque/type-visitor.cc
    ../../third_party/v8/src/torque/type-visitor.h
    ../../third_party/v8/src/torque/types.cc
    ../../third_party/v8/src/torque/types.h
    ../../third_party/v8/src/torque/utils.cc
    ../../third_party/v8/src/torque/utils.h
    ../../third_party/v8/src/tracing/trace-event.cc
    ../../third_party/v8/src/tracing/trace-event.h
    ../../third_party/v8/src/tracing/traced-value.cc
    ../../third_party/v8/src/tracing/traced-value.h
    ../../third_party/v8/src/tracing/tracing-category-observer.cc
    ../../third_party/v8/src/tracing/tracing-category-observer.h
    ../../third_party/v8/src/transitions-inl.h
    ../../third_party/v8/src/transitions.cc
    ../../third_party/v8/src/transitions.h
    ../../third_party/v8/src/trap-handler/handler-inside-posix.cc
    ../../third_party/v8/src/trap-handler/handler-inside-posix.h
    ../../third_party/v8/src/trap-handler/handler-inside.cc
    ../../third_party/v8/src/trap-handler/handler-outside-posix.cc
    ../../third_party/v8/src/trap-handler/handler-outside.cc
    ../../third_party/v8/src/trap-handler/handler-shared.cc
    ../../third_party/v8/src/trap-handler/trap-handler-internal.h
    ../../third_party/v8/src/trap-handler/trap-handler.h
    ../../third_party/v8/src/turbo-assembler.cc
    ../../third_party/v8/src/turbo-assembler.h
    ../../third_party/v8/src/type-hints.cc
    ../../third_party/v8/src/type-hints.h
    ../../third_party/v8/src/type-traits.h
    ../../third_party/v8/src/unicode-decoder.cc
    ../../third_party/v8/src/unicode-decoder.h
    ../../third_party/v8/src/unicode-inl.h
    ../../third_party/v8/src/unicode.cc
    ../../third_party/v8/src/unicode.h
    ../../third_party/v8/src/unoptimized-compilation-info.cc
    ../../third_party/v8/src/unoptimized-compilation-info.h
    ../../third_party/v8/src/unwinder.cc
    ../../third_party/v8/src/uri.cc
    ../../third_party/v8/src/uri.h
    ../../third_party/v8/src/utils-inl.h
    ../../third_party/v8/src/utils.cc
    ../../third_party/v8/src/utils.h
    ../../third_party/v8/src/utils/address-map.h
    ../../third_party/v8/src/utils/allocation.h
    ../../third_party/v8/src/utils/boxed-float.h
    ../../third_party/v8/src/utils/memcopy.h
    ../../third_party/v8/src/utils/pointer-with-payload.h
    ../../third_party/v8/src/utils/utils-inl.h
    ../../third_party/v8/src/utils/utils.h
    ../../third_party/v8/src/utils/vector.h
    ../../third_party/v8/src/v8.cc
    ../../third_party/v8/src/v8.h
    ../../third_party/v8/src/v8memory.h
    ../../third_party/v8/src/v8threads.cc
    ../../third_party/v8/src/v8threads.h
    ../../third_party/v8/src/value-serializer.cc
    ../../third_party/v8/src/value-serializer.h
    ../../third_party/v8/src/vector-slot-pair.cc
    ../../third_party/v8/src/vector-slot-pair.h
    ../../third_party/v8/src/vector.h
    ../../third_party/v8/src/version.cc
    ../../third_party/v8/src/version.h
    ../../third_party/v8/src/visitors.cc
    ../../third_party/v8/src/visitors.h
    ../../third_party/v8/src/vm-state-inl.h
    ../../third_party/v8/src/vm-state.h
    ../../third_party/v8/src/wasm/baseline/liftoff-assembler-defs.h
    ../../third_party/v8/src/wasm/baseline/liftoff-assembler.cc
    ../../third_party/v8/src/wasm/baseline/liftoff-assembler.h
    ../../third_party/v8/src/wasm/baseline/liftoff-compiler.cc
    ../../third_party/v8/src/wasm/baseline/liftoff-compiler.h
    ../../third_party/v8/src/wasm/baseline/liftoff-register.h
    ../../third_party/v8/src/wasm/baseline/x64/liftoff-assembler-x64.h
    ../../third_party/v8/src/wasm/compilation-environment.h
    ../../third_party/v8/src/wasm/decoder.h
    ../../third_party/v8/src/wasm/function-body-decoder-impl.h
    ../../third_party/v8/src/wasm/function-body-decoder.cc
    ../../third_party/v8/src/wasm/function-body-decoder.h
    ../../third_party/v8/src/wasm/function-compiler.cc
    ../../third_party/v8/src/wasm/function-compiler.h
    ../../third_party/v8/src/wasm/graph-builder-interface.cc
    ../../third_party/v8/src/wasm/graph-builder-interface.h
    ../../third_party/v8/src/wasm/js-to-wasm-wrapper-cache.h
    ../../third_party/v8/src/wasm/jump-table-assembler.cc
    ../../third_party/v8/src/wasm/jump-table-assembler.h
    ../../third_party/v8/src/wasm/leb-helper.h
    ../../third_party/v8/src/wasm/local-decl-encoder.cc
    ../../third_party/v8/src/wasm/local-decl-encoder.h
    ../../third_party/v8/src/wasm/memory-tracing.cc
    ../../third_party/v8/src/wasm/memory-tracing.h
    ../../third_party/v8/src/wasm/module-compiler.cc
    ../../third_party/v8/src/wasm/module-compiler.h
    ../../third_party/v8/src/wasm/module-decoder.cc
    ../../third_party/v8/src/wasm/module-decoder.h
    ../../third_party/v8/src/wasm/module-instantiate.cc
    ../../third_party/v8/src/wasm/module-instantiate.h
    ../../third_party/v8/src/wasm/object-access.h
    ../../third_party/v8/src/wasm/signature-map.cc
    ../../third_party/v8/src/wasm/signature-map.h
    ../../third_party/v8/src/wasm/streaming-decoder.cc
    ../../third_party/v8/src/wasm/streaming-decoder.h
    ../../third_party/v8/src/wasm/value-type.h
    ../../third_party/v8/src/wasm/wasm-code-manager.cc
    ../../third_party/v8/src/wasm/wasm-code-manager.h
    ../../third_party/v8/src/wasm/wasm-constants.h
    ../../third_party/v8/src/wasm/wasm-debug.cc
    ../../third_party/v8/src/wasm/wasm-engine.cc
    ../../third_party/v8/src/wasm/wasm-engine.h
    ../../third_party/v8/src/wasm/wasm-external-refs.cc
    ../../third_party/v8/src/wasm/wasm-external-refs.h
    ../../third_party/v8/src/wasm/wasm-feature-flags.h
    ../../third_party/v8/src/wasm/wasm-features.cc
    ../../third_party/v8/src/wasm/wasm-features.h
    ../../third_party/v8/src/wasm/wasm-import-wrapper-cache.cc
    ../../third_party/v8/src/wasm/wasm-import-wrapper-cache.h
    ../../third_party/v8/src/wasm/wasm-interpreter.cc
    ../../third_party/v8/src/wasm/wasm-interpreter.h
    ../../third_party/v8/src/wasm/wasm-js.cc
    ../../third_party/v8/src/wasm/wasm-js.h
    ../../third_party/v8/src/wasm/wasm-limits.h
    ../../third_party/v8/src/wasm/wasm-linkage.h
    ../../third_party/v8/src/wasm/wasm-memory.cc
    ../../third_party/v8/src/wasm/wasm-memory.h
    ../../third_party/v8/src/wasm/wasm-module-builder.cc
    ../../third_party/v8/src/wasm/wasm-module-builder.h
    ../../third_party/v8/src/wasm/wasm-module.cc
    ../../third_party/v8/src/wasm/wasm-module.h
    ../../third_party/v8/src/wasm/wasm-objects-inl.h
    ../../third_party/v8/src/wasm/wasm-objects.cc
    ../../third_party/v8/src/wasm/wasm-objects.h
    ../../third_party/v8/src/wasm/wasm-opcodes.cc
    ../../third_party/v8/src/wasm/wasm-opcodes.h
    ../../third_party/v8/src/wasm/wasm-result.cc
    ../../third_party/v8/src/wasm/wasm-result.h
    ../../third_party/v8/src/wasm/wasm-serialization.cc
    ../../third_party/v8/src/wasm/wasm-serialization.h
    ../../third_party/v8/src/wasm/wasm-text.cc
    ../../third_party/v8/src/wasm/wasm-text.h
    ../../third_party/v8/src/wasm/wasm-tier.h
    ../../third_party/v8/src/wasm/wasm-value.h
    ../../third_party/v8/src/x64/assembler-x64-inl.h
    ../../third_party/v8/src/x64/assembler-x64.cc
    ../../third_party/v8/src/x64/assembler-x64.h
    ../../third_party/v8/src/x64/constants-x64.h
    ../../third_party/v8/src/x64/cpu-x64.cc
    ../../third_party/v8/src/x64/deoptimizer-x64.cc
    ../../third_party/v8/src/x64/disasm-x64.cc
    ../../third_party/v8/src/x64/eh-frame-x64.cc
    ../../third_party/v8/src/x64/frame-constants-x64.cc
    ../../third_party/v8/src/x64/frame-constants-x64.h
    ../../third_party/v8/src/x64/interface-descriptors-x64.cc
    ../../third_party/v8/src/x64/macro-assembler-x64.cc
    ../../third_party/v8/src/x64/macro-assembler-x64.h
    ../../third_party/v8/src/x64/register-x64.h
    ../../third_party/v8/src/x64/sse-instr.h
    ../../third_party/v8/src/zone/accounting-allocator.cc
    ../../third_party/v8/src/zone/accounting-allocator.h
    ../../third_party/v8/src/zone/zone-allocator.h
    ../../third_party/v8/src/zone/zone-chunk-list.h
    ../../third_party/v8/src/zone/zone-containers.h
    ../../third_party/v8/src/zone/zone-handle-set.h
    ../../third_party/v8/src/zone/zone-list-inl.h
    ../../third_party/v8/src/zone/zone-segment.cc
    ../../third_party/v8/src/zone/zone-segment.h
    ../../third_party/v8/src/zone/zone-splay-tree.h
    ../../third_party/v8/src/zone/zone.cc
    ../../third_party/v8/src/zone/zone.h
    ../../third_party/v8/third_party/inspector_protocol/encoding/encoding.cc
    ../../third_party/v8/third_party/inspector_protocol/encoding/encoding.h
    gen/core/libdeno/libdeno_jumbo_1.cc
    gen/third_party/v8/builtins-generated/bytecodes-builtins-list.h
    gen/third_party/v8/debug-support.cc
    gen/third_party/v8/embedded.S
    gen/third_party/v8/extras-libraries.cc
    gen/third_party/v8/include/inspector/Debugger.h
    gen/third_party/v8/include/inspector/Runtime.h
    gen/third_party/v8/include/inspector/Schema.h
    gen/third_party/v8/snapshot.cc
    gen/third_party/v8/src/inspector/inspector_jumbo_1.cc
    gen/third_party/v8/src/inspector/protocol/Console.cpp
    gen/third_party/v8/src/inspector/protocol/Console.h
    gen/third_party/v8/src/inspector/protocol/Debugger.cpp
    gen/third_party/v8/src/inspector/protocol/Debugger.h
    gen/third_party/v8/src/inspector/protocol/Forward.h
    gen/third_party/v8/src/inspector/protocol/HeapProfiler.cpp
    gen/third_party/v8/src/inspector/protocol/HeapProfiler.h
    gen/third_party/v8/src/inspector/protocol/Profiler.cpp
    gen/third_party/v8/src/inspector/protocol/Profiler.h
    gen/third_party/v8/src/inspector/protocol/Protocol.cpp
    gen/third_party/v8/src/inspector/protocol/Protocol.h
    gen/third_party/v8/src/inspector/protocol/Runtime.cpp
    gen/third_party/v8/src/inspector/protocol/Runtime.h
    gen/third_party/v8/src/inspector/protocol/Schema.cpp
    gen/third_party/v8/src/inspector/protocol/Schema.h
    gen/third_party/v8/torque-generated/builtin-definitions-from-dsl.h
    gen/third_party/v8/torque-generated/builtin-definitions-tq.h
    gen/third_party/v8/torque-generated/builtins-arguments-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-arguments-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-copywithin-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-copywithin-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-filter-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-filter-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-find-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-find-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-findindex-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-findindex-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-foreach-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-foreach-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-join-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-join-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-lastindexof-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-lastindexof-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-map-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-map-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-of-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-of-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-reverse-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-reverse-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-shift-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-shift-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-slice-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-slice-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-splice-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-splice-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-array-unshift-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-array-unshift-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-base-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-base-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-collections-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-collections-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-data-view-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-data-view-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-extras-utils-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-extras-utils-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-growable-fixed-array-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-growable-fixed-array-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-internal-coverage-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-internal-coverage-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-iterator-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-iterator-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-object-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-object-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-proxy-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-proxy-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-regexp-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-regexp-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-regexp-replace-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-regexp-replace-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-string-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-string-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-string-html-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-string-html-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-string-repeat-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-string-repeat-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-test-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-test-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-createtypedarray-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-createtypedarray-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-every-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-every-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-filter-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-filter-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-find-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-find-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-findindex-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-findindex-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-foreach-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-foreach-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-reduce-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-reduce-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-reduceright-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-reduceright-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-slice-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-slice-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-some-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-some-from-dsl-gen.h
    gen/third_party/v8/torque-generated/builtins-typed-array-subarray-from-dsl-gen.cc
    gen/third_party/v8/torque-generated/builtins-typed-array-subarray-from-dsl-gen.h
    gen/third_party/v8/torque-generated/class-definitions-from-dsl.h
    gen/third_party/v8/torque-generated/class-definitions-tq-inl.h
    gen/third_party/v8/torque-generated/class-definitions-tq.cc
    gen/third_party/v8/torque-generated/class-definitions-tq.h
    gen/third_party/v8/torque-generated/class-verifiers-tq.cc
    gen/third_party/v8/torque-generated/class-verifiers-tq.h
    gen/third_party/v8/torque-generated/field-offsets-tq.h
    gen/third_party/v8/torque-generated/objects-printer-tq.cc
    gen/third_party/v8/torque_base_jumbo_1.cc
    gen/third_party/v8/torque_generated_definitions_jumbo_1.cc
    gen/third_party/v8/torque_generated_initializers_jumbo_1.cc
    gen/third_party/v8/v8_base_without_compiler_jumbo_1.cc
    gen/third_party/v8/v8_base_without_compiler_jumbo_2.cc
    gen/third_party/v8/v8_base_without_compiler_jumbo_3.cc
    gen/third_party/v8/v8_base_without_compiler_jumbo_4.cc
    gen/third_party/v8/v8_base_without_compiler_jumbo_5.cc
    gen/third_party/v8/v8_base_without_compiler_jumbo_6.cc
    gen/third_party/v8/v8_base_without_compiler_jumbo_7.cc
    gen/third_party/v8/v8_base_without_compiler_jumbo_8.cc
    gen/third_party/v8/v8_compiler_jumbo_1.cc
    gen/third_party/v8/v8_compiler_jumbo_2.cc
    gen/third_party/v8/v8_compiler_jumbo_3.cc
    gen/third_party/v8/v8_initializers_jumbo_1.cc
    gen/third_party/v8/v8_nosnapshot_jumbo_1.cc
    gen/third_party/v8/v8_snapshot_jumbo_1.cc
```

Reply:
That's ~1300 files more than you have in your list